<!doctype html>
<html lang="en">
<head>
	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="description" content="Constraint-based Metabolic Modeling in Python.">
	<meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
	<meta name="generator" content="Hugo 0.59.0" />
	<title>cobrapy - constraint-based metabolic modeling in Python</title>

    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" type="text/css">
	<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
	<link rel="stylesheet" href="https://storage.googleapis.com/code.getmdl.io/1.3.0/material.teal-indigo.min.css">
	<link rel="stylesheet" href="../css/styles.css">
    <script defer src="https://code.getmdl.io/1.3.0/material.min.js"></script>
    <link rel="icon" type="image/png" href="../media/favicon.png" />
</head>
<body>
<div class="mdl-layout mdl-js-layout mdl-layout--fixed-header">
<header class="mdl-layout__header mdl-shadow--6dp mdl-color-text--black">
<div class="mdl-layout__header-row mdl-color-text--black">
	<span class="mdl-layout-title">Menu</span>
	<div class="mdl-layout-spacer"></div>
	<nav class="mdl-navigation mdl-layout--large-screen-only">
		<a class="mdl-navigation__link mdl-color-text--black" href="../">Home</a>
        <a class="mdl-navigation__link" href="../releases">Releases</a>
		<a class="mdl-navigation__link mdl-color-text--black" href="../packages">Packages</a>
        <a class="mdl-navigation__link mdl-color-text--black" href="../pubs">Publications</a>
        <a class="mdl-navigation__link mdl-color-text--black" href="https://cobrapy.readthedocs.io/en/stable/">Docs</a>
	</nav>
</div>
</header>
<div class="mdl-layout__drawer">
	<span class="mdl-layout-title"><img src="../media/logo.png" width="100%"></span>
	<nav class="mdl-navigation">
        <a class="mdl-navigation__link" href="../">Home</a>
        <a class="mdl-navigation__link" href="../releases">Releases</a>
		<a class="mdl-navigation__link" href="../packages">Packages</a>
        <a class="mdl-navigation__link" href="../pubs">Publications</a>
        <a class="mdl-navigation__link" href="https://cobrapy.readthedocs.io/en/stable/">Docs</a>
	</nav>
</div>
<main class="mdl-layout__content">
<div class="page-content mdl-grid">

<div class="mdl-cell mdl-cell--12-col">
    <div class="mdl-card mdl-shadow--4dp">
        <div class="mdl-card__title mdl-color--indigo-900" style="height:auto">
            <div class="mdl-card__title-text mdl-color-text--white">Publications</div>
        </div>
    </div>
</div>
<div class="mdl-tabs mdl-js-tabs mdl-js-ripple-effect">
    <div class="mdl-tabs__tab-bar">
        
        <a href="#year_2019" class="mdl-tabs__tab is-active">2019</a>
        
        <a href="#year_2018" class="mdl-tabs__tab ">2018</a>
        
        <a href="#year_2017" class="mdl-tabs__tab ">2017</a>
        
        <a href="#year_2016" class="mdl-tabs__tab ">2016</a>
        
        <a href="#year_2015" class="mdl-tabs__tab ">2015</a>
        
        <a href="#year_2014" class="mdl-tabs__tab ">2014</a>
        
        <a href="#year_2013" class="mdl-tabs__tab ">2013</a>
        
    </div>
    
    <div class="mdl-tabs__panel is-active" id="year_2019">
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">SAMMI: A Semi-Automated Tool for the Visualization of Metabolic Networks.</h1>
                    <h3 class="mdl-color-text--gray header">Bioinformatics (Oxford, England)&nbsp;2019</h3>
                    
                    <h2 class="mdl-color-text--primary header">Andre Schultz and Rehan Akbani</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Here we present a browser based Semi-Automated Metabolic Map Illustrator (SAMMI) for the visualization of metabolic networks. While automated features allow for easy network partitioning, navigation, and node positioning, SAMMI also offers a wide array of manual map editing features. This combination allows for fast, context specific visualization of metabolic networks as well as the development of standardized, large-scale, visually appealing maps. The implementation of SAMMI with popular constraint-based modeling toolboxes also allows for effortless visualization of simulation results of genome-scale metabolic models.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1093/bioinformatics/btz927">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/31851289">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Effects of MCHM on yeast metabolism.</h1>
                    <h3 class="mdl-color-text--gray header">PloS One&nbsp;2019</h3>
                    
                    <h2 class="mdl-color-text--primary header">Amaury Pupo, Kang Mo Ku and Jennifer E G Gallagher</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>On January 2014 approximately 10,000 gallons of crude 4-Methylcyclohexanemethanol (MCHM) and propylene glycol phenol ether (PPH) were accidentally released into the Elk River, West Virginia, contaminating the tap water of around 300,000 residents. Crude MCHM is an industrial chemical used as flotation reagent to clean coal. At the time of the spill, MCHM&rsquo;s toxicological data were limited, an issue that has been addressed by different studies focused on understanding the immediate and long-term effects of MCHM on human health and the environment.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pone.0223909">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/31622418">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Evolution and regulation of nitrogen flux through compartmentalized metabolic networks in a marine diatom.</h1>
                    <h3 class="mdl-color-text--gray header">Nature Communications&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Sarah R Smith, Chris L Dupont, James K McCarthy, Jared T Broddrick, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Diatoms outcompete other phytoplankton for nitrate, yet little is known about the mechanisms underpinning this ability. Genomes and genome-enabled studies have shown that diatoms possess unique features of nitrogen metabolism however, the implications for nutrient utilization and growth are poorly understood. Using a combination of transcriptomics, proteomics, metabolomics, fluxomics, and flux balance analysis to examine short-term shifts in nitrogen utilization in the model pennate diatom in Phaeodactylum tricornutum, we obtained a systems-level understanding of assimilation and intracellular distribution of nitrogen.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1038/s41467-019-12407-y">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/31591397">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">RetSynth: determining all optimal and sub-optimal synthetic pathways that facilitate synthesis of target compounds in chassis organisms.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Bioinformatics&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Leanne S Whitmore, Bernard Nguyen, Ali Pinar, Anthe George, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Flux Balance Analysis, Metabolic Engineering, Mixed Integer Linear Programming</p>
                    <p>The efficient biological production of industrially and economically important compounds is a challenging problem. Brute-force determination of the optimal pathways to efficient production of a target chemical in a chassis organism is computationally intractable. Many current methods provide a single solution to this problem, but fail to provide all optimal pathways, optional sub-optimal solutions or hybrid biological/non-biological solutions.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s12859-019-3025-9">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/31500573">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">p13CMFA: Parsimonious 13C metabolic flux analysis.</h1>
                    <h3 class="mdl-color-text--gray header">PLoS Computational Biology&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Carles Foguet, Anusha Jayaraman, Silvia Marin, Vitaly A Selivanov, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Deciphering the mechanisms of regulation of metabolic networks subjected to perturbations, including disease states and drug-induced stress, relies on tracing metabolic fluxes. One of the most informative data to predict metabolic fluxes are 13C based metabolomics, which provide information about how carbons are redistributed along central carbon metabolism. Such data can be integrated using 13C Metabolic Flux Analysis (13C MFA) to provide quantitative metabolic maps of flux distributions. However, 13C MFA might be unable to reduce the solution space towards a unique solution either in large metabolic networks or when small sets of measurements are integrated.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pcbi.1007310">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/31490922">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Gene connectivity and enzyme evolution in the human metabolic network.</h1>
                    <h3 class="mdl-color-text--gray header">Biology Direct&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Begoña Dobon, Ludovica Montanucci, Juli Peretó, Jaume Bertranpetit, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Connectivity, Degree, Enzymes, Metabolism, Network Topology, Positive Selection, Purifying Selection</p>
                    <p>Determining the factors involved in the likelihood of a gene being under adaptive selection is still a challenging goal in Evolutionary Biology. Here, we perform an evolutionary analysis of the human metabolic genes to explore the associations between network structure and the presence and strength of natural selection in the genes whose products are involved in metabolism. Purifying and positive selection are estimated at interspecific (among mammals) and intraspecific (among human populations) levels, and the connections between enzymatic reactions are differentiated between incoming (in-degree) and outgoing (out-degree) links.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s13062-019-0248-7">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/31481097">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Flux sampling is a powerful tool to study metabolism under changing environmental conditions.</h1>
                    <h3 class="mdl-color-text--gray header">NPJ Systems Biology and Applications&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Helena A Herrmann, Beth C Dyson, Lucy Vass, Giles N Johnson, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Biochemical Networks, Plant Sciences</p>
                    <p>The development of high-throughput &lsquo;omic techniques has sparked a rising interest in genome-scale metabolic models, with applications ranging from disease diagnostics to crop adaptation. Efficient and accurate methods are required to analyze large metabolic networks. Flux sampling can be used to explore the feasible flux solutions in metabolic networks by generating probability distributions of steady-state reaction fluxes. Unlike other methods, flux sampling can be used without assuming a particular cellular objective.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1038/s41540-019-0109-0">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/31482008">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">A systems biology approach for studying Wolbachia metabolism reveals points of interaction with its host in the context of arboviral infection.</h1>
                    <h3 class="mdl-color-text--gray header">PLoS Neglected Tropical Diseases&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Natalia E Jiménez, Ziomara P Gerdtzen, Álvaro Olivera-Nappa, J Cristian Salgado, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Wolbachia are alpha-proteobacteria known to infect arthropods, which are of interest for disease control since they have been associated with improved resistance to viral infection. Although several genomes for different strains have been sequenced, there is little knowledge regarding the relationship between this bacterium and their hosts, particularly on their dependency for survival. Motivated by the potential applications on disease control, we developed genome-scale models of four Wolbachia strains known to infect arthropods: wAlbB (Aedes albopictus), wVitA (Nasonia vitripennis), wMel and wMelPop (Drosophila melanogaster).</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pntd.0007678">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/31469838">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Systems Biology and Pangenome of <i>Salmonella</i> O-Antigens.</h1>
                    <h3 class="mdl-color-text--gray header">MBio&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Yara Seif, Jonathan M Monk, Henrique Machado, Erol Kavvas, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Salmonella, Genome Analysis, Metabolism, Serogroups</p>
                    <p>O-antigens are glycopolymers in lipopolysaccharides expressed on the cell surface of Gram-negative bacteria. Variability in the O-antigen structure constitutes the basis for the establishment of the serotyping schema. We pursued a two-pronged approach to define the basis for O-antigen structural diversity. First, we developed a bottom-up systems biology approach to O-antigen metabolism by building a reconstruction of</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1128/mBio.01247-19">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/31455646">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Simulation of heterosis in a genome-scale metabolic network provides mechanistic explanations for increased biomass production rates in hybrid plants.</h1>
                    <h3 class="mdl-color-text--gray header">NPJ Systems Biology and Applications&nbsp;2019</h3>
                    
                    <h2 class="mdl-color-text--primary header">Michael Vacher and Ian Small</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Computer Modelling, Systems Analysis</p>
                    <p>Heterosis, or hybrid vigour, is said to occur when F1 individuals exhibit increased performance for a number of traits compared to their parental lines. Improved traits can include increased size, better yield, faster development and a higher tolerance to pathogens or adverse conditions. The molecular basis for the phenomenon remains disputed, despite many decades of theorising and experimentation. In this study, we add a genetics layer to a constraint-based model of plant (</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1038/s41540-019-0101-8">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/31341636">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">FindTargetsWEB: A User-Friendly Tool for Identification of Potential Therapeutic Targets in Metabolic Networks of Bacteria.</h1>
                    <h3 class="mdl-color-text--gray header">Frontiers in Genetics&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Thiago Castanheira Merigueti, Marcia Weber Carneiro, Ana Paula D&#39;A Carvalho-Assef, Floriano Paes Silva-Jr, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        COBRA Analysis, Python (Programming Language), Flux Balance Analysis, Metabolic Network, Systems Biology</p>
                    <p>None</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.3389/fgene.2019.00633">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/31333719">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">A mass and charge balanced metabolic model of Setaria viridis revealed mechanisms of proton balancing in C4 plants.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Bioinformatics&nbsp;2019</h3>
                    
                    <h2 class="mdl-color-text--primary header">Rahul Shaw and C Y Maurice Cheung</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Ammonium and Nitrate Usage, Bioenergy Grasses, C4 Photosynthesis, Gene Association, Genome-Scale Metabolic Network Model, Lignocellulosic Biomass, Mass and Charge Balance, Millet, Setaria Viridis</p>
                    <p>C4 photosynthesis is a key domain of plant research with outcomes ranging from crop quality improvement, biofuel production and efficient use of water and nutrients. A metabolic network model of C4 &ldquo;lab organism&rdquo; Setaria viridis with extensive gene-reaction associations can accelerate target identification for desired metabolic manipulations and thereafter in vivo validation. Moreover, metabolic reconstructions have also been shown to be a significant tool to investigate fundamental metabolic traits.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s12859-019-2941-z">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/31248364">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Leaf Energy Balance Requires Mitochondrial Respiration and Export of Chloroplast NADPH in the Light.</h1>
                    <h3 class="mdl-color-text--gray header">Plant Physiology&nbsp;2019</h3>
                    
                    <h2 class="mdl-color-text--primary header">Sanu Shameer, R George Ratcliffe and Lee J Sweetlove</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Key aspects of leaf mitochondrial metabolism in the light remain unresolved. For example, there is debate about the relative importance of exporting reducing equivalents from mitochondria for the peroxisomal steps of photorespiration versus oxidation of NADH to generate ATP by oxidative phosphorylation. Here, we address this and explore energetic coupling between organelles in the light using a diel flux balance analysis model. The model included more than 600 reactions of central metabolism with full stoichiometric accounting of energy production and consumption.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1104/pp.19.00624">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/31213510">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Identification of critical connectors in the directed reaction-centric graphs of microbial metabolic networks.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Bioinformatics&nbsp;2019</h3>
                    
                    <h2 class="mdl-color-text--primary header">Eun-Youn Kim, Daniel Ashlock and Sung Ho Yoon</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Cascade Number, Centrality Metric, Directed Network, Information Flow, Metabolic Network, Reaction-Centric Graph</p>
                    <p>Detection of central nodes in asymmetrically directed biological networks depends on centrality metrics quantifying individual nodes&rsquo; importance in a network. In topological analyses on metabolic networks, various centrality metrics have been mostly applied to metabolite-centric graphs. However, centrality metrics including those not depending on high connections are largely unexplored for directed reaction-centric graphs.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s12859-019-2897-z">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/31195955">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Human Systems Biology and Metabolic Modelling: A Review-From Disease Metabolism to Precision Medicine.</h1>
                    <h3 class="mdl-color-text--gray header">BioMed Research International&nbsp;2019</h3>
                    
                    <h2 class="mdl-color-text--primary header">Claudio Angione</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>In cell and molecular biology, metabolism is the only system that can be fully simulated at genome scale. Metabolic systems biology offers powerful abstraction tools to simulate all known metabolic reactions in a cell, therefore providing a snapshot that is close to its observable phenotype. In this review, we cover the 15 years of human metabolic modelling. We show that, although the past five years have not experienced large improvements in the size of the gene and metabolite sets in human metabolic models, their accuracy is rapidly increasing.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1155/2019/8304260">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/31281846">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Reconstruction of a Genome Scale Metabolic Model of the polyhydroxybutyrate producing methanotroph Methylocystis parvus OBBP.</h1>
                    <h3 class="mdl-color-text--gray header">Microbial Cell Factories&nbsp;2019</h3>
                    
                    <h2 class="mdl-color-text--primary header">Sergio Bordel, Antonia Rojas and Raúl Muñoz</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Genome-Scale Metabolic Models, Metabolism, Methanotrophs, Methylocystis</p>
                    <p>Methylocystis parvus is a type II methanotroph characterized by its high specific methane degradation rate (compared to other methanotrophs of the same family) and its ability to accumulate up to 50% of its biomass in the form of poly-3-hydroxybutyrate (PHB) under nitrogen limiting conditions. This makes it a very promising cell factory.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s12934-019-1154-5">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/31170985">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">A Genome-Scale Metabolic Model of Soybean (<i>Glycine max</i>) Highlights Metabolic Fluxes in Seedlings.</h1>
                    <h3 class="mdl-color-text--gray header">Plant Physiology&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Thiago Batista Moreira, Rahul Shaw, Xinyu Luo, Oishik Ganguly, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Until they become photoautotrophic juvenile plants, seedlings depend upon the reserves stored in seed tissues. These reserves must be mobilized and metabolized, and their breakdown products must be distributed to the different organs of the growing seedling. Here, we investigated the mobilization of soybean (</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1104/pp.19.00122">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/31171578">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">BOFdat: Generating biomass objective functions for genome-scale metabolic models from experimental data.</h1>
                    <h3 class="mdl-color-text--gray header">PLoS Computational Biology&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Jean-Christophe Lachance, Colton J Lloyd, Jonathan M Monk, Laurence Yang, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Genome-scale metabolic models (GEMs) are mathematically structured knowledge bases of metabolism that provide phenotypic predictions from genomic information. GEM-guided predictions of growth phenotypes rely on the accurate definition of a biomass objective function (BOF) that is designed to include key cellular biomass components such as the major macromolecules (DNA, RNA, proteins), lipids, coenzymes, inorganic ions and species-specific components. Despite its importance, no standardized computational platform is currently available to generate species-specific biomass objective functions in a data-driven, unbiased fashion.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pcbi.1006971">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/31009451">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Core Metabolism Shifts during Growth on Methanol versus Methane in the Methanotroph <i>Methylomicrobium buryatense</i> 5GB1.</h1>
                    <h3 class="mdl-color-text--gray header">MBio&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Yanfen Fu, Lian He, Jennifer Reeve, David A C Beck, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        13C Tracer Analysis, Flux Balance Analysis, Methanol, Methanotrophs</p>
                    <p>None</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1128/mBio.00406-19">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30967465">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Enzyme promiscuity shapes adaptation to novel growth substrates.</h1>
                    <h3 class="mdl-color-text--gray header">Molecular Systems Biology&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Gabriela I Guzmán, Troy E Sandberg, Ryan A LaCroix, Ákos Nyerges, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Adaptive Evolution, Enzyme Promiscuity, Genome‐scale Modeling, Systems Biology</p>
                    <p>Evidence suggests that novel enzyme functions evolved from low-level promiscuous activities in ancestral enzymes. Yet, the evolutionary dynamics and physiological mechanisms of how such side activities contribute to systems-level adaptations are not well characterized. Furthermore, it remains untested whether knowledge of an organism&rsquo;s promiscuous reaction set, or underground metabolism, can aid in forecasting the genetic basis of metabolic adaptations. Here, we employ a computational model of underground metabolism and laboratory evolution experiments to examine the role of enzyme promiscuity in the acquisition and optimization of growth on predicted non-native substrates in</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.15252/msb.20188462">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30962359">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Approaches to Computational Strain Design in the Multiomics Era.</h1>
                    <h3 class="mdl-color-text--gray header">Frontiers in Microbiology&nbsp;2019</h3>
                    
                    <h2 class="mdl-color-text--primary header">Peter C St John and Yannick J Bomble</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Constraint-Based Methods, Kinetic Metabolic Models, Machine Learning, Multiomics, Strain Engineering</p>
                    <p>Modern omics analyses are able to effectively characterize the genetic, regulatory, and metabolic phenotypes of engineered microbes, yet designing genetic interventions to achieve a desired phenotype remains challenging. With recent developments in genetic engineering techniques, timelines associated with building and testing strain designs have been greatly reduced, allowing for the first time an efficient closed loop iteration between experiment and analysis. However, the scale and complexity associated with multi-omics datasets complicates manual biological reasoning about the mechanisms driving phenotypic changes.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.3389/fmicb.2019.00597">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/31024467">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Dataset of differential gene expression between total normal human thyroid and histologically normal thyroid adjacent to papillary thyroid carcinoma.</h1>
                    <h3 class="mdl-color-text--gray header">Data in Brief&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Lorenza Vitale, Allison Piovesan, Francesca Antonaros, Pierluigi Strippoli, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>This article contains further data and information from our published manuscript [1]. We aim to identify significant transcriptome alterations of total normal human thyroid vs. histologically normal thyroid adjacent to papillary thyroid carcinoma. We performed a systematic meta-analysis of all the available gene expression profiles for the whole organ also collecting gene expression data for the normal thyroid adjacent to papillary thyroid carcinoma. A differential quantitative transcriptome reference map was generated by using TRAM (Transcriptome Mapper) software able to combine, normalize and integrate a total of 35 datasets from total normal thyroid and 40 datasets from histologically normal thyroid adjacent to papillary thyroid carcinoma from different sources.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1016/j.dib.2019.103835">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/31049370">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Coupling S-adenosylmethionine-dependent methylation to growth: Design and uses.</h1>
                    <h3 class="mdl-color-text--gray header">PLoS Biology&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Hao Luo, Anne Sofie L Hansen, Lei Yang, Konstantin Schneider, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>We present a selection design that couples S-adenosylmethionine-dependent methylation to growth. We demonstrate its use in improving the enzyme activities of not only N-type and O-type methyltransferases by 2-fold but also an acetyltransferase of another enzyme category when linked to a methylation pathway in Escherichia coli using adaptive laboratory evolution. We also demonstrate its application for drug discovery using a catechol O-methyltransferase and its inhibitors entacapone and tolcapone. Implementation of this design in Saccharomyces cerevisiae is also demonstrated.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pbio.2007050">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30856169">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Predicting proteome allocation, overflow metabolism, and metal requirements in a model acetogen.</h1>
                    <h3 class="mdl-color-text--gray header">PLoS Computational Biology&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Joanne K Liu, Colton Lloyd, Mahmoud M Al-Bassam, Ali Ebrahim, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>The unique capability of acetogens to ferment a broad range of substrates renders them ideal candidates for the biotechnological production of commodity chemicals. In particular the ability to grow with H2:CO2 or syngas (a mixture of H2/CO/CO2) makes these microorganisms ideal chassis for sustainable bioproduction. However, advanced design strategies for acetogens are currently hampered by incomplete knowledge about their physiology and our inability to accurately predict phenotypes. Here we describe the reconstruction of a novel genome-scale model of metabolism and macromolecular synthesis (ME-model) to gain new insights into the biology of the model acetogen Clostridium ljungdahlii.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pcbi.1006848">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30845144">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">The genetic basis for adaptation of model-designed syntrophic co-cultures.</h1>
                    <h3 class="mdl-color-text--gray header">PLoS Computational Biology&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Colton J Lloyd, Zachary A King, Troy E Sandberg, Ying Hefner, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Understanding the fundamental characteristics of microbial communities could have far reaching implications for human health and applied biotechnology. Despite this, much is still unknown regarding the genetic basis and evolutionary strategies underlying the formation of viable synthetic communities. By pairing auxotrophic mutants in co-culture, it has been demonstrated that viable nascent E. coli communities can be established where the mutant strains are metabolically coupled. A novel algorithm, OptAux, was constructed to design 61 unique multi-knockout E.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pcbi.1006213">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30822347">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Maximum entropy and population heterogeneity in continuous cell cultures.</h1>
                    <h3 class="mdl-color-text--gray header">PLoS Computational Biology&nbsp;2019</h3>
                    
                    <h2 class="mdl-color-text--primary header">Jorge Fernandez-de-Cossio-Diaz and Roberto Mulet</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Continuous cultures of mammalian cells are complex systems displaying hallmark phenomena of nonlinear dynamics, such as multi-stability, hysteresis, as well as sharp transitions between different metabolic states. In this context mathematical models may suggest control strategies to steer the system towards desired states. Although even clonal populations are known to exhibit cell-to-cell variability, most of the currently studied models assume that the population is homogeneous. To overcome this limitation, we use the maximum entropy principle to model the phenotypic distribution of cells in a chemostat as a function of the dilution rate.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pcbi.1006823">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30811392">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Creation and analysis of biochemical constraint-based models using the COBRA Toolbox v.3.0.</h1>
                    <h3 class="mdl-color-text--gray header">Nature Protocols&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Laurent Heirendt, Sylvain Arreckx, Thomas Pfau, Sebastián N Mendoza, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Constraint-based reconstruction and analysis (COBRA) provides a molecular mechanistic framework for integrative analysis of experimental molecular systems biology data and quantitative prediction of physicochemically and biochemically feasible phenotypic states. The COBRA Toolbox is a comprehensive desktop software suite of interoperable COBRA methods. It has found widespread application in biology, biomedicine, and biotechnology because its functions can be flexibly combined to implement tailored COBRA protocols for any biochemical network. This protocol is an update to the COBRA Toolbox v.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1038/s41596-018-0098-2">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30787451">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Gsmodutils: a python based framework for test-driven genome scale metabolic model development.</h1>
                    <h3 class="mdl-color-text--gray header">Bioinformatics (Oxford, England)&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">James Gilbert, Nicole Pearcy, Rupert Norman, Thomas Millat, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Genome scale metabolic models (GSMMs) are increasingly important for systems biology and metabolic engineering research as they are capable of simulating complex steady-state behaviour. Constraints based models of this form can include thousands of reactions and metabolites, with many crucial pathways that only become activated in specific simulation settings. However, despite their widespread use, power and the availability of tools to aid with the construction and analysis of large scale models, little methodology is suggested for their continued management.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1093/bioinformatics/btz088">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30759197">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Cross-compartment metabolic coupling enables flexible photoprotective mechanisms in the diatom Phaeodactylum tricornutum.</h1>
                    <h3 class="mdl-color-text--gray header">The New Phytologist&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Jared T Broddrick, Niu Du, Sarah R Smith, Yoshinori Tsuji, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Phaeodactylum Tricornutum, Analysis, Diatom, Energy Metabolism, Flux Balance, Genome-Scale Modeling, Photorespiration</p>
                    <p>Photoacclimation consists of short- and long-term strategies used by photosynthetic organisms to adapt to dynamic light environments. Observable photophysiology changes resulting from these strategies have been used in coarse-grained models to predict light-dependent growth and photosynthetic rates. However, the contribution of the broader metabolic network, relevant to species-specific strategies and fitness, is not accounted for in these simple models. We incorporated photophysiology experimental data with genome-scale modeling to characterize organism-level, light-dependent metabolic changes in the model diatom Phaeodactylum tricornutum.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1111/nph.15685">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30636322">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Multi-Omics and Genome-Scale Modeling Reveal a Metabolic Shift During <i>C. elegans</i> Aging.</h1>
                    <h3 class="mdl-color-text--gray header">Frontiers in Molecular Biosciences&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Janna Hastings, Abraham Mains, Bhupinder Virk, Nicolas Rodriguez, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        C. Elegans, Aging, Flux Balance Analysis, Metabolomics, Multi-Omics, Systems Biology, Whole Genome Model</p>
                    <p>In this contribution, we describe a multi-omics systems biology study of the metabolic changes that occur during aging in</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.3389/fmolb.2019.00002">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30788345">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">ModelExplorer - software for visual inspection and inconsistency correction of genome-scale metabolic reconstructions.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Bioinformatics&nbsp;2019</h3>
                    
                    <h2 class="mdl-color-text--primary header">Nikolay Martyushenko and Eivind Almaas</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Consistency Checking, Constraint Based Modeling, FBA, Metabolic Model, Network Visualization</p>
                    <p>Genome-scale metabolic network reconstructions are low level chemical representations of biological organisms. These models allow the system-level investigation of metabolic phenotypes using a variety of computational approaches. The link between a metabolic network model and an organisms&rsquo; higher-level behaviour is usually found using a constraint-based analysis approach, such as FBA (Flux Balance Analysis). However, the process of model reconstruction rarely proceeds without error. Often, considerable parts of a model cannot carry flux under any condition.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s12859-019-2615-x">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30691403">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Essential metabolism for a minimal cell.</h1>
                    <h3 class="mdl-color-text--gray header">ELife&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Marian Breuer, Tyler M Earnest, Chuck Merryman, Kim S Wise, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        JCVI-Syn3A, Computational Biology, Gene Essentiality, Metabolic Reconstruction, Mycoplasma, Proteomics, Systems Biology, Transposon Mutagenesis</p>
                    <p>JCVI-syn3A, a robust minimal cell with a 543 kbp genome and 493 genes, provides a versatile platform to study the basics of life. Using the vast amount of experimental information available on its precursor,</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.7554/eLife.36842">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30657448">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Machine learning framework for assessment of microbial factory performance.</h1>
                    <h3 class="mdl-color-text--gray header">PloS One&nbsp;2019</h3>
                    
                    <h2 class="mdl-color-text--primary header">Tolutola Oyetunde, Di Liu, Hector Garcia Martin and Yinjie J Tang</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Metabolic models can estimate intrinsic product yields for microbial factories, but such frameworks struggle to predict cell performance (including product titer or rate) under suboptimal metabolism and complex bioprocess conditions. On the other hand, machine learning, complementary to metabolic modeling necessitates large amounts of data. Building such a database for metabolic engineering designs requires significant manpower and is prone to human errors and bias. We propose an approach to integrate data-driven methods with genome scale metabolic model for assessment of microbial bio-production (yield, titer and rate).</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pone.0210558">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30645629">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Emergent Subpopulation Behavior Uncovered with a Community Dynamic Metabolic Model of <i>Escherichia coli</i> Diauxic Growth.</h1>
                    <h3 class="mdl-color-text--gray header">MSystems&nbsp;2019</h3>
                    
                    <h2 class="mdl-color-text--primary header">Antonella Succurro, Daniel Segrè and Oliver Ebenhöh</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Diauxic Growth, Metabolic Network Modeling, Microbial Communities, Population Heterogeneity</p>
                    <p>Microbes have adapted to greatly variable environments in order to survive both short-term perturbations and permanent changes. A classical and yet still actively studied example of adaptation to dynamic environments is the diauxic shift of Escherichia coli, in which cells grow on glucose until its exhaustion and then transition to using previously secreted acetate. Here we tested different hypotheses concerning the nature of this transition by using dynamic metabolic modeling. To reach this goal, we developed an open source modeling framework integrating dynamic models (ordinary differential equation systems) with structural models (metabolic networks) which can take into account the behavior of multiple subpopulations and smooth flux transitions between time points.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1128/mSystems.00230-18">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30944873">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">A computational knowledge-base elucidates the response of Staphylococcus aureus to different media types.</h1>
                    <h3 class="mdl-color-text--gray header">PLoS Computational Biology&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Yara Seif, Jonathan M Monk, Nathan Mih, Hannah Tsunemoto, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>S. aureus is classified as a serious threat pathogen and is a priority that guides the discovery and development of new antibiotics. Despite growing knowledge of S. aureus metabolic capabilities, our understanding of its systems-level responses to different media types remains incomplete. Here, we develop a manually reconstructed genome-scale model (GEM-PRO) of metabolism with 3D protein structures for S. aureus USA300 str. JE2 containing 854 genes, 1,440 reactions, 1,327 metabolites and 673 3-dimensional protein structures.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pcbi.1006644">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30625152">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">An interspecies malate-pyruvate shuttle reconciles redox imbalance in an anaerobic microbial community.</h1>
                    <h3 class="mdl-color-text--gray header">The ISME Journal&nbsp;2019</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Po-Hsiang Wang, Kevin Correia, Han-Chen Ho, Naveen Venayak, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Microbes in ecosystems often develop coordinated metabolic interactions. Therefore, understanding metabolic interdependencies between microbes is critical to deciphering ecosystem function. In this study, we sought to deconstruct metabolic interdependencies in organohalide-respiring consortium ACT-3 containing Dehalobacter restrictus using a combination of metabolic modeling and experimental validation. D. restrictus possesses a complete set of genes for amino acid biosynthesis yet when grown in isolation requires amino acid supplementation. We reconciled this discrepancy using flux balance analysis considering cofactor availability, enzyme promiscuity, and shared protein expression patterns for several D.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1038/s41396-018-0333-4">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30607026">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <h6>Total of 36 publications. 😄</h6>
        </div>
    </div>
    
    <div class="mdl-tabs__panel " id="year_2018">
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">pyTFA and matTFA: a Python package and a Matlab toolbox for Thermodynamics-based Flux Analysis.</h1>
                    <h3 class="mdl-color-text--gray header">Bioinformatics (Oxford, England)&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Pierre Salvy, Georgios Fengos, Meric Ataman, Thomas Pathier, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>pyTFA and matTFA are the first published implementations of the original TFA paper. Specifically, they include explicit formulation of Gibbs energies and metabolite concentrations, which enables straightforward integration of metabolite concentration measurements.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1093/bioinformatics/bty499">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30561545">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Reframing gene essentiality in terms of adaptive flexibility.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Systems Biology&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Gabriela I Guzmán, Connor A Olson, Ying Hefner, Patrick V Phaneuf, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Adaptive Evolution, Essentiality, Genome-Scale Model</p>
                    <p>Essentiality assays are important tools commonly utilized for the discovery of gene functions. Growth/no growth screens of single gene knockout strain collections are also often utilized to test the predictive power of genome-scale models. False positive predictions occur when computational analysis predicts a gene to be non-essential, however experimental screens deem the gene to be essential. One explanation for this inconsistency is that the model contains the wrong information, possibly an incorrectly annotated alternative pathway or isozyme reaction.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s12918-018-0653-z">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30558585">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">MoVE identifies metabolic valves to switch between phenotypic states.</h1>
                    <h3 class="mdl-color-text--gray header">Nature Communications&nbsp;2018</h3>
                    
                    <h2 class="mdl-color-text--primary header">Naveen Venayak, Axel von Kamp, Steffen Klamt and Radhakrishnan Mahadevan</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Metabolism is highly regulated, allowing for robust and complex behavior. This behavior can often be achieved by controlling a small number of important metabolic reactions, or metabolic valves. Here, we present a method to identify the location of such valves: the metabolic valve enumerator (MoVE). MoVE uses a metabolic model to identify genetic intervention strategies which decouple two desired phenotypes. We apply this method to identify valves which can decouple growth and production to systematically improve the rate and yield of biochemical production processes.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1038/s41467-018-07719-4">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30552335">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">A genome-scale metabolic model of potato late blight suggests a photosynthesis suppression mechanism.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Genomics&nbsp;2018</h3>
                    
                    <h2 class="mdl-color-text--primary header">Kelly Botero, Silvia Restrepo and Andres Pinzón</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Compatible Interaction, Flux Balance Analysis, Metabolic Reconstruction, Phytophthora Infestans, Solanum Tuberosum, Systems Biology</p>
                    <p>Phytophthora infestans is a plant pathogen that causes an important plant disease known as late blight in potato plants (Solanum tuberosum) and several other solanaceous hosts. This disease is the main factor affecting potato crop production worldwide. In spite of the importance of the disease, the molecular mechanisms underlying the compatibility between the pathogen and its hosts are still unknown.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s12864-018-5192-x">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30537923">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">A Genome-Scale Metabolic Model for <i>Methylococcus capsulatus</i> (Bath) Suggests Reduced Efficiency Electron Transfer to the Particulate Methane Monooxygenase.</h1>
                    <h3 class="mdl-color-text--gray header">Frontiers in Microbiology&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Christian Lieven, Leander A H Petersen, Sten Bay Jørgensen, Krist V Gernaey, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        C1 Metabolism, COBRA, Constraint-Based Reconstruction and Analysis, Genome-Scale Metabolic Reconstruction, Methanotrophy, Single Cell Protein</p>
                    <p>None</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.3389/fmicb.2018.02947">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30564208">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">FLYCOP: metabolic modeling-based analysis and engineering microbial communities.</h1>
                    <h3 class="mdl-color-text--gray header">Bioinformatics (Oxford, England)&nbsp;2018</h3>
                    
                    <h2 class="mdl-color-text--primary header">Beatriz García-Jiménez, José Luis García and Juan Nogales</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Synthetic microbial communities begin to be considered as promising multicellular biocatalysts having a large potential to replace engineered single strains in biotechnology applications, in pharmaceutical, chemical and living architecture sectors. In contrast to single strain engineering, the effective and high-throughput analysis and engineering of microbial consortia face the lack of knowledge, tools and well-defined workflows. This manuscript contributes to fill this important gap with a framework, called FLYCOP (FLexible sYnthetic Consortium OPtimization), which contributes to microbial consortia modeling and engineering, while improving the knowledge about how these communities work.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1093/bioinformatics/bty561">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30423096">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Predicting the metabolic capabilities of Synechococcus elongatus PCC 7942 adapted to different light regimes.</h1>
                    <h3 class="mdl-color-text--gray header">Metabolic Engineering&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Jared T Broddrick, David G Welkie, Denis Jallet, Susan S Golden, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Constraint Based Modeling, Cyanobacteria Engineering, Flux Balance Analysis, Genome-Scale Modeling, Photosynthesis, Synechococcus Elongatus</p>
                    <p>There is great interest in engineering photoautotrophic metabolism to generate bioproducts of societal importance. Despite the success in employing genome-scale modeling coupled with flux balance analysis to engineer heterotrophic metabolism, the lack of proper constraints necessary to generate biologically realistic predictions has hindered broad application of this methodology to phototrophic metabolism. Here we describe a methodology for constraining genome-scale models of photoautotrophy in the cyanobacteria Synechococcus elongatus PCC 7942. Experimental photophysiology parameters coupled to genome-scale flux balance analysis resulted in accurate predictions of growth rates and metabolic reaction fluxes at low and high light conditions.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1016/j.ymben.2018.11.001">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30439494">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">The EcoCyc Database.</h1>
                    <h3 class="mdl-color-text--gray header">EcoSal Plus&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Peter D Karp, Wai Kit Ong, Suzanne Paley, Richard Billington, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>EcoCyc is a bioinformatics database available at EcoCyc.org that describes the genome and the biochemical machinery of</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1128/ecosalplus.ESP-0006-2018">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30406744">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Metagenomics-Based, Strain-Level Analysis of <i>Escherichia coli</i> From a Time-Series of Microbiome Samples From a Crohn's Disease Patient.</h1>
                    <h3 class="mdl-color-text--gray header">Frontiers in Microbiology&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Xin Fang, Jonathan M Monk, Sergey Nurk, Margarita Akseshina, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Escherichia Coli, De Novo Assembly, Gut Microbiome, Inflammatory Bowel Disease, Metagenomics</p>
                    <p>Dysbiosis of the gut microbiome, including elevated abundance of putative leading bacterial triggers such as</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.3389/fmicb.2018.02559">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30425690">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">RAVEN 2.0: A versatile toolbox for metabolic network reconstruction and a case study on Streptomyces coelicolor.</h1>
                    <h3 class="mdl-color-text--gray header">PLoS Computational Biology&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Hao Wang, Simonas Marcišauskas, Benjamín J Sánchez, Iván Domenzain, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>RAVEN is a commonly used MATLAB toolbox for genome-scale metabolic model (GEM) reconstruction, curation and constraint-based modelling and simulation. Here we present RAVEN Toolbox 2.0 with major enhancements, including: (i) de novo reconstruction of GEMs based on the MetaCyc pathway database; (ii) a redesigned KEGG-based reconstruction pipeline; (iii) convergence of reconstructions from various sources; (iv) improved performance, usability, and compatibility with the COBRA Toolbox. Capabilities of RAVEN 2.0 are here illustrated through de novo reconstruction of GEMs for the antibiotic-producing bacterium Streptomyces coelicolor.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pcbi.1006541">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30335785">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Identification of growth-coupled production strains considering protein costs and kinetic variability.</h1>
                    <h3 class="mdl-color-text--gray header">Metabolic Engineering Communications&nbsp;2018</h3>
                    
                    <h2 class="mdl-color-text--primary header">Hoang V Dinh, Zachary A King, Bernhard O Palsson and Adam M Feist</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Conversion of renewable biomass to useful molecules in microbial cell factories can be approached in a rational and systematic manner using constraint-based reconstruction and analysis. Filtering for high confidence</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1016/j.mec.2018.e00080">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30370222">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Escher-FBA: a web application for interactive flux balance analysis.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Systems Biology&nbsp;2018</h3>
                    
                    <h2 class="mdl-color-text--primary header">Elliot Rowe, Bernhard O Palsson and Zachary A King</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Constraint-Based Modeling, Escher, Flux Balance Analysis, Metabolism, Visualization, Web Application</p>
                    <p>Flux balance analysis (FBA) is a widely-used method for analyzing metabolic networks. However, most existing tools that implement FBA require downloading software and writing code. Furthermore, FBA generates predictions for metabolic networks with thousands of components, so meaningful changes in FBA solutions can be difficult to identify. These challenges make it difficult for beginners to learn how FBA works.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s12918-018-0607-5">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30257674">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">A community-driven reconstruction of the <i>Aspergillus niger</i> metabolic network.</h1>
                    <h3 class="mdl-color-text--gray header">Fungal Biology and Biotechnology&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Julian Brandl, Maria Victoria Aguilar-Pontes, Paul Schäpe, Anders Noerregaard, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Aspergillus Niger, Genome-Scale Model, Primary Metabolism, Secondary Metabolism</p>
                    <p>None</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s40694-018-0060-7">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30275963">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Role of phosphate limitation and pyruvate decarboxylase in rewiring of the metabolic network for increasing flux towards isoprenoid pathway in a TATA binding protein mutant of Saccharomyces cerevisiae.</h1>
                    <h3 class="mdl-color-text--gray header">Microbial Cell Factories&nbsp;2018</h3>
                    
                    <h2 class="mdl-color-text--primary header">Manisha Wadhwa, Sumana Srinivasan, Anand K Bachhawat and K V Venkatesh</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Isoprenoid Pathway, Metabolic Flux Distribution, NADPH, PDC6, Phosphate, SPT15</p>
                    <p>Production of isoprenoids, a large and diverse class of commercially important chemicals, can be achieved through engineering metabolism in microorganisms. Several attempts have been made to reroute metabolic flux towards isoprenoid pathway in yeast. Most approaches have focused on the core isoprenoid pathway as well as on meeting the increased precursors and cofactor requirements. To identify unexplored genetic targets that positively influence the isoprenoid pathway activity, a carotenoid based genetic screen was previously developed and three novel mutants of a global TATA binding protein SPT15 was isolated for heightened isoprenoid flux in Saccharomyces cerevisiae.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s12934-018-1000-1">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30241525">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Metabolic network reconstruction and phenome analysis of the industrial microbe, Escherichia coli BL21(DE3).</h1>
                    <h3 class="mdl-color-text--gray header">PloS One&nbsp;2018</h3>
                    
                    <h2 class="mdl-color-text--primary header">Hanseol Kim, Sinyeon Kim and Sung Ho Yoon</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Escherichia coli BL21(DE3) is an industrial model microbe for the mass-production of bioproducts such as biofuels, biorefineries, and recombinant proteins. However, despite its important role in scientific research and biotechnological applications, a high-quality metabolic network model for metabolic engineering is yet to be developed. Here, we present the comprehensive metabolic network model of E. coli BL21(DE3), named iHK1487, based on the latest genome reannotation and phenome analysis. The metabolic model consists of 1,164 unique metabolites, 2,701 metabolic reactions, and 1,487 genes.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pone.0204375">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30240424">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Inferring Metabolic Mechanisms of Interaction within a Defined Gut Microbiota.</h1>
                    <h3 class="mdl-color-text--gray header">Cell Systems&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Gregory L Medlock, Maureen A Carey, Dennis G McDuffie, Michael B Mundy, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Emergent Behavior, Interspecies Interactions, Mathematical Modeling, Metabolic Cross-Feeding, Metabolism, Microbial Community, Microbial Ecology, Microbiome, Microbiota, Systems Biology</p>
                    <p>The diversity and number of species present within microbial communities create the potential for a multitude of interspecies metabolic interactions. Here, we develop, apply, and experimentally test a framework for inferring metabolic mechanisms associated with interspecies interactions. We perform pairwise growth and metabolome profiling of co-cultures of strains from a model mouse microbiota. We then apply our framework to dissect emergent metabolic behaviors that occur in co-culture. Based on one of the inferences from this framework, we identify and interrogate an amino acid cross-feeding interaction and validate that the proposed interaction leads to a growth benefit in vitro.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1016/j.cels.2018.08.003">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30195437">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">A Prospective Study on the Fermentation Landscape of Gaseous Substrates to Biorenewables Using <i>Methanosarcina acetivorans</i> Metabolic Model.</h1>
                    <h3 class="mdl-color-text--gray header">Frontiers in Microbiology&nbsp;2018</h3>
                    
                    <h2 class="mdl-color-text--primary header">Hadi Nazem-Bokaee and Costas D Maranas</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        CH4, CO, CO2, M. Acetivorans, Gas Fermentation, Metabolic Modeling</p>
                    <p>The abundance of methane in shale gas and of other gases such as carbon monoxide, hydrogen, and carbon dioxide as chemical process byproducts has motivated the use of gas fermentation for bioproduction. Recent advances in metabolic engineering and synthetic biology allow for engineering of microbes metabolizing a variety of chemicals including gaseous feeds into a number of biorenewables and transportation liquid fuels. New computational tools enable the systematic exploration of all feasible conversion alternatives.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.3389/fmicb.2018.01855">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30197630">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">An enormous potential for niche construction through bacterial cross-feeding in a homogeneous environment.</h1>
                    <h3 class="mdl-color-text--gray header">PLoS Computational Biology&nbsp;2018</h3>
                    
                    <h2 class="mdl-color-text--primary header">Magdalena San Roman and Andreas Wagner</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Microorganisms modify their environment by excreting by-products of metabolism, which can create new ecological niches that can help microbial populations diversify. A striking example comes from experimental evolution of genetically identical Escherichia coli populations that are grown in a homogeneous environment with the single carbon source glucose. In such experiments, stable communities of genetically diverse cross-feeding E. coli cells readily emerge. Some cells that consume the primary carbon source glucose excrete a secondary carbon source, such as acetate, that sustains other community members.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pcbi.1006340">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/30040834">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">COBRAme: A computational framework for genome-scale models of metabolism and gene expression.</h1>
                    <h3 class="mdl-color-text--gray header">PLoS Computational Biology&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Colton J Lloyd, Ali Ebrahim, Laurence Yang, Zachary A King, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Genome-scale models of metabolism and macromolecular expression (ME-models) explicitly compute the optimal proteome composition of a growing cell. ME-models expand upon the well-established genome-scale models of metabolism (M-models), and they enable a new fundamental understanding of cellular growth. ME-models have increased predictive capabilities and accuracy due to their inclusion of the biosynthetic costs for the machinery of life, but they come with a significant increase in model size and complexity. This challenge results in models which are both difficult to compute and challenging to understand conceptually.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pcbi.1006302">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29975681">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Development and validation of an updated computational model of Streptomyces coelicolor primary and secondary metabolism.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Genomics&nbsp;2018</h3>
                    
                    <h2 class="mdl-color-text--primary header">Adam Amara, Eriko Takano and Rainer Breitling</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Genome-Scale Metabolic Modelling, Natural Products, Omics, Secondary Metabolism, Streptomyces, Synthetic Biology</p>
                    <p>Streptomyces species produce a vast diversity of secondary metabolites of clinical and biotechnological importance, in particular antibiotics. Recent developments in metabolic engineering, synthetic and systems biology have opened new opportunities to exploit Streptomyces secondary metabolism, but achieving industry-level production without time-consuming optimization has remained challenging. Genome-scale metabolic modelling has been shown to be a powerful tool to guide metabolic engineering strategies for accelerated strain optimization, and several generations of models of Streptomyces metabolism have been developed for this purpose.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s12864-018-4905-5">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29973148">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">A Dynamic Multi-Tissue Flux Balance Model Captures Carbon and Nitrogen Metabolism and Optimal Resource Partitioning During Arabidopsis Growth.</h1>
                    <h3 class="mdl-color-text--gray header">Frontiers in Plant Science&nbsp;2018</h3>
                    
                    <h2 class="mdl-color-text--primary header">Rahul Shaw and C Y Maurice Cheung</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Dynamic Flux Balance Analysis, Genome-Scale Metabolic Modeling, Multi-Tissue Model, Resource Allocation, Seedling Growth</p>
                    <p>Plant metabolism is highly adapted in response to its surrounding for acquiring limiting resources. In this study, a dynamic flux balance modeling framework with a multi-tissue (leaf and root) diel genome-scale metabolic model of</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.3389/fpls.2018.00884">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29997643">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">DMPy: a Python package for automated mathematical model construction of large-scale metabolic systems.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Systems Biology&nbsp;2018</h3>
                    
                    <h2 class="mdl-color-text--primary header">Robert W Smith, Rik P van Rosmalen, Vitor A P Martins Dos Santos and Christian Fleck</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Automated Data Collection, Constraint-Based Metabolic Models, Dynamic Mathematical Model, Genome-Scale, Parameter Optimisation</p>
                    <p>Models of metabolism are often used in biotechnology and pharmaceutical research to identify drug targets or increase the direct production of valuable compounds. Due to the complexity of large metabolic systems, a number of conclusions have been drawn using mathematical methods with simplifying assumptions. For example, constraint-based models describe changes of internal concentrations that occur much quicker than alterations in cell physiology. Thus, metabolite concentrations and reaction fluxes are fixed to constant values.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s12918-018-0584-8">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29914475">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Tellurium notebooks-An environment for reproducible dynamical modeling in systems biology.</h1>
                    <h3 class="mdl-color-text--gray header">PLoS Computational Biology&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">J Kyle Medley, Kiri Choi, Matthias König, Lucian Smith, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>The considerable difficulty encountered in reproducing the results of published dynamical models limits validation, exploration and reuse of this increasingly large biomedical research resource. To address this problem, we have developed Tellurium Notebook, a software system for model authoring, simulation, and teaching that facilitates building reproducible dynamical models and reusing models by 1) providing a notebook environment which allows models, Python code, and narrative to be intermixed, 2) supporting the COMBINE archive format during model development for capturing model information in an exchangeable format and 3) enabling users to easily simulate and edit public COMBINE-compliant models from public repositories to facilitate studying model dynamics, variants and test cases.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pcbi.1006220">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29906293">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Escherichia coli B2 strains prevalent in inflammatory bowel disease patients have distinct metabolic capabilities that enable colonization of intestinal mucosa.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Systems Biology&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Xin Fang, Jonathan M Monk, Nathan Mih, Bin Du, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Inflammatory Bowel Disease, Metabolic Modeling, Pan-Genome Analysis</p>
                    <p>Escherichia coli is considered a leading bacterial trigger of inflammatory bowel disease (IBD). E. coli isolates from IBD patients primarily belong to phylogroup B2. Previous studies have focused on broad comparative genomic analysis of E. coli B2 isolates, and identified virulence factors that allow B2 strains to reside within human intestinal mucosa. Metabolic capabilities of E. coli strains have been shown to be related to their colonization site, but remain unexplored in IBD-associated strains.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s12918-018-0587-5">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29890970">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Traceability, reproducibility and wiki-exploration for "à-la-carte" reconstructions of genome-scale metabolic models.</h1>
                    <h3 class="mdl-color-text--gray header">PLoS Computational Biology&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Méziane Aite, Marie Chevallier, Clémence Frioux, Camille Trottier, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Genome-scale metabolic models have become the tool of choice for the global analysis of microorganism metabolism, and their reconstruction has attained high standards of quality and reliability. Improvements in this area have been accompanied by the development of some major platforms and databases, and an explosion of individual bioinformatics methods. Consequently, many recent models result from &ldquo;à la carte&rdquo; pipelines, combining the use of platforms, individual tools and biological expertise to enhance the quality of the reconstruction.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pcbi.1006146">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29791443">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Constraint based modeling of metabolism allows finding metabolic cancer hallmarks and identifying personalized therapeutic windows.</h1>
                    <h3 class="mdl-color-text--gray header">Oncotarget&nbsp;2018</h3>
                    
                    <h2 class="mdl-color-text--primary header">Sergio Bordel</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        RNA-Seq, Flux Balance Analysis, Metabolic Models, Metabolism, Therapeutic Windows</p>
                    <p>In order to choose optimal personalized anticancer treatments, transcriptomic data should be analyzed within the frame of biological networks. The best known human biological network (in terms of the interactions between its different components) is metabolism. Cancer cells have been known to have specific metabolic features for a long time and currently there is a growing interest in characterizing new cancer specific metabolic hallmarks. In this article it is presented a method to find personalized therapeutic windows using RNA-seq data and Genome Scale Metabolic Models.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.18632/oncotarget.24805">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29731977">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">iCN718, an Updated and Improved Genome-Scale Metabolic Network Reconstruction of <i>Acinetobacter baumannii</i> AYE.</h1>
                    <h3 class="mdl-color-text--gray header">Frontiers in Genetics&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Charles J Norsigian, Erol Kavvas, Yara Seif, Bernhard O Palsson, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Acinetobacter Baumannii, Antibiotic Resistance, Constraint-Based Modeling, Genome-Scale Reconstruction, Metabolism</p>
                    <p>None</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.3389/fgene.2018.00121">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29692801">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Updated and standardized genome-scale reconstruction of Mycobacterium tuberculosis H37Rv, iEK1011, simulates flux states indicative of physiological conditions.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Systems Biology&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Erol S Kavvas, Yara Seif, James T Yurkovich, Charles Norsigian, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Antibiotic Resistance, Environmental Condition, Genome-Scale Reconstruction, M. Tuberculosis</p>
                    <p>The efficacy of antibiotics against M. tuberculosis has been shown to be influenced by experimental media conditions. Investigations of M. tuberculosis growth in physiological conditions have described an environment that is different from common in vitro media. Thus, elucidating the interplay between available nutrient sources and antibiotic efficacy has clear medical relevance. While genome-scale reconstructions of M. tuberculosis have enabled the ability to interrogate media differences for the past 10 years, recent reconstructions have diverged from each other without standardization.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s12918-018-0557-y">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29499714">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Review of Statistical Learning Methods in Integrated Omics Studies (An Integrated Information Science).</h1>
                    <h3 class="mdl-color-text--gray header">Bioinformatics and Biology Insights&nbsp;2018</h3>
                    
                    <h2 class="mdl-color-text--primary header">Irene Sui Lan Zeng and Thomas Lumley</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Statistical Learnings, Exploratory Learning, Integrated Omics, Network Learning, Regression</p>
                    <p>Integrated omics is becoming a new channel for investigating the complex molecular system in modern biological science and sets a foundation for systematic learning for precision medicine. The statistical/machine learning methods that have emerged in the past decade for integrated omics are not only innovative but also multidisciplinary with integrated knowledge in biology, medicine, statistics, machine learning, and artificial intelligence. Here, we review the nontrivial classes of learning methods from the statistical aspects and streamline these learning methods within the statistical learning framework.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1177/1177932218759292">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29497285">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">ssbio: a Python framework for structural systems biology.</h1>
                    <h3 class="mdl-color-text--gray header">Bioinformatics (Oxford, England)&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Nathan Mih, Elizabeth Brunk, Ke Chen, Edward Catoiu, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Working with protein structures at the genome-scale has been challenging in a variety of ways. Here, we present ssbio, a Python package that provides a framework to easily work with structural information in the context of genome-scale network reconstructions, which can contain thousands of individual proteins. The ssbio package provides an automated pipeline to construct high quality genome-scale models with protein structures (GEM-PROs), wrappers to popular third-party programs to compute associated protein properties, and methods to visualize and annotate structures directly in Jupyter notebooks, thus lowering the barrier of linking 3D structural data with established systems workflows.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1093/bioinformatics/bty077">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29444205">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Plasma and urinary metabolomic profiles of Down syndrome correlate with alteration of mitochondrial metabolism.</h1>
                    <h3 class="mdl-color-text--gray header">Scientific Reports&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Maria Caracausi, Veronica Ghini, Chiara Locatelli, Martina Mericio, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Down syndrome (DS) is caused by the presence of a supernumerary copy of the human chromosome 21 (Hsa21) and is the most frequent genetic cause of intellectual disability (ID). Key traits of DS are the distinctive facies and cognitive impairment. We conducted for the first time an analysis of the Nuclear Magnetic Resonance (NMR)-detectable part of the metabolome in plasma and urine samples, studying 67 subjects with DS and 29 normal subjects as controls selected among DS siblings.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1038/s41598-018-20834-y">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29445163">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Systems assessment of transcriptional regulation on central carbon metabolism by Cra and CRP.</h1>
                    <h3 class="mdl-color-text--gray header">Nucleic Acids Research&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Donghyuk Kim, Sang Woo Seo, Ye Gao, Hojung Nam, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Two major transcriptional regulators of carbon metabolism in bacteria are Cra and CRP. CRP is considered to be the main mediator of catabolite repression. Unlike for CRP, in vivo DNA binding information of Cra is scarce. Here we generate and integrate ChIP-exo and RNA-seq data to identify 39 binding sites for Cra and 97 regulon genes that are regulated by Cra in Escherichia coli. An integrated metabolic-regulatory network was formed by including experimentally-derived regulatory information and a genome-scale metabolic network reconstruction.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1093/nar/gky069">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29394395">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Prediction of reaction knockouts to maximize succinate production by Actinobacillus succinogenes.</h1>
                    <h3 class="mdl-color-text--gray header">PloS One&nbsp;2018</h3>
                    
                    <h2 class="mdl-color-text--primary header">Ambarish Nag, Peter C St John, Michael F Crowley and Yannick J Bomble</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Succinate is a precursor of multiple commodity chemicals and bio-based succinate production is an active area of industrial bioengineering research. One of the most important microbial strains for bio-based production of succinate is the capnophilic gram-negative bacterium Actinobacillus succinogenes, which naturally produces succinate by a mixed-acid fermentative pathway. To engineer A. succinogenes to improve succinate yields during mixed acid fermentation, it is important to have a detailed understanding of the metabolic flux distribution in A.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pone.0189144">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29381705">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Genome-wide characterization of Phytophthora infestans metabolism: a systems biology approach.</h1>
                    <h3 class="mdl-color-text--gray header">Molecular Plant Pathology&nbsp;2018</h3>
                    
                    <h2 class="mdl-color-text--primary header">Sander Y A Rodenburg, Michael F Seidl, Dick de Ridder and Francine Govers</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Phytophthora Infestans, Metabolic Model, Metabolism, Oomycete, Systems Biology</p>
                    <p>Genome-scale metabolic models (GEMs) provide a functional view of the complex network of biochemical reactions in the living cell. Initially mainly applied to reconstruct the metabolism of model organisms, the availability of increasingly sophisticated reconstruction methods and more extensive biochemical databases now make it possible to reconstruct GEMs for less well-characterized organisms, and have the potential to unravel the metabolism in pathogen-host systems. Here, we present a GEM for the oomycete plant pathogen Phytophthora infestans as a first step towards an integrative model with its host.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1111/mpp.12623">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/28990716">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Model-driven design of a minimal medium for Akkermansia muciniphila confirms mucus adaptation.</h1>
                    <h3 class="mdl-color-text--gray header">Microbial Biotechnology&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Kees C H van der Ark, Steven Aalvink, Maria Suarez-Diez, Peter J Schaap, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>The abundance of the human intestinal symbiont Akkermansia muciniphila has found to be inversely correlated with several diseases, including metabolic syndrome and obesity. A. muciniphila is known to use mucin as sole carbon and nitrogen source. To study the physiology and the potential for therapeutic applications of this bacterium, we designed a defined minimal medium. The composition of the medium was based on the genome-scale metabolic model of A. muciniphila and the composition of mucin.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1111/1751-7915.13033">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29377524">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">One-step fermentative production of aromatic polyesters from glucose by metabolically engineered Escherichia coli strains.</h1>
                    <h3 class="mdl-color-text--gray header">Nature Communications&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Jung Eun Yang, Si Jae Park, Won Jun Kim, Hyeong Jun Kim, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Aromatic polyesters are widely used plastics currently produced from petroleum. Here we engineer Escherichia coli strains for the production of aromatic polyesters from glucose by one-step fermentation. When the Clostridium difficile isocaprenoyl-CoA:2-hydroxyisocaproate CoA-transferase (HadA) and evolved polyhydroxyalkanoate (PHA) synthase genes are overexpressed in a D-phenyllactate-producing strain, poly(52.3 mol% 3-hydroxybutyrate (3HB)-co-47.7 mol% D-phenyllactate) can be produced from glucose and sodium 3HB. Also, various poly(3HB-co-D-phenyllactate) polymers having 11.0, 15.8, 20.0, 70.8, and 84.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1038/s41467-017-02498-w">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29311546">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Genome scale metabolic models as tools for drug design and personalized medicine.</h1>
                    <h3 class="mdl-color-text--gray header">PloS One&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Vytautas Raškevičius, Valeryia Mikalayeva, Ieva Antanavičiūtė, Ieva Ceslevičienė, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>In this work we aim to show how Genome Scale Metabolic Models (GSMMs) can be used as tools for drug design. By comparing the chemical structures of human metabolites (obtained using their KEGG indexes) and the compounds contained in the DrugBank database, we have observed that compounds showing Tanimoto scores higher than 0.9 with a metabolite, are 29.5 times more likely to bind the enzymes metabolizing the considered metabolite, than ligands chosen randomly.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pone.0190636">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29304175">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Constraining Genome-Scale Models to Represent the Bow Tie Structure of Metabolism for <sup>13</sup>C Metabolic Flux Analysis.</h1>
                    <h3 class="mdl-color-text--gray header">Metabolites&nbsp;2018</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Tyler W H Backman, David Ando, Jahnavi Singh, Jay D Keasling, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        13C Metabolic Flux Analysis, Cellular Metabolism, Flux Balance Analysis, Genome Scale Models, Linear Programming, Simulated Annealing, Stoichiometry, Two-Scale 13C Metabolic Flux Analysis</p>
                    <p>Determination of internal metabolic fluxes is crucial for fundamental and applied biology because they map how carbon and electrons flow through metabolism to enable cell function. 13 C Metabolic Flux Analysis ( 13 C MFA) and Two-Scale 13 C Metabolic Flux Analysis (2S- 13 C MFA) are two techniques used to determine such fluxes. Both operate on the simplifying approximation that metabolic flux from peripheral metabolism into central &ldquo;core&rdquo; carbon metabolism is minimal, and can be omitted when modeling isotopic labeling in core metabolism.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.3390/metabo8010003">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29300340">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <h6>Total of 38 publications. 😄</h6>
        </div>
    </div>
    
    <div class="mdl-tabs__panel " id="year_2017">
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Emerging whole-cell modeling principles and methods.</h1>
                    <h3 class="mdl-color-text--gray header">Current Opinion in Biotechnology&nbsp;2017</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Arthur P Goldberg, Balázs Szigeti, Yin Hoon Chew, John Ap Sekar, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Whole-cell computational models aim to predict cellular phenotypes from genotype by representing the entire genome, the structure and concentration of each molecular species, each molecular interaction, and the extracellular environment. Whole-cell models have great potential to transform bioscience, bioengineering, and medicine. However, numerous challenges remain to achieve whole-cell models. Nevertheless, researchers are beginning to leverage recent progress in measurement technology, bioinformatics, data sharing, rule-based modeling, and multi-algorithmic simulation to build the first whole-cell models.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1016/j.copbio.2017.12.013">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29275251">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Analysis of <i>Piscirickettsia salmonis</i> Metabolism Using Genome-Scale Reconstruction, Modeling, and Testing.</h1>
                    <h3 class="mdl-color-text--gray header">Frontiers in Microbiology&nbsp;2017</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">María P Cortés, Sebastián N Mendoza, Dante Travisany, Alexis Gaete, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Piscirickettsia, Constraint-Based, Genome-Scale, Metabolism, Pathogen, Salmonis</p>
                    <p>None</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.3389/fmicb.2017.02462">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29321769">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header"><i>In Silico</i> Analysis of the Small Molecule Content of Outer Membrane Vesicles Produced by <i>Bacteroides thetaiotaomicron</i> Indicates an Extensive Metabolic Link between Microbe and Host.</h1>
                    <h3 class="mdl-color-text--gray header">Frontiers in Microbiology&nbsp;2017</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">William A Bryant, Régis Stentz, Gwenaelle Le Gall, Michael J E Sternberg, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Bacteroides Thetaiotaomicron VPI-5482, Genome-Scale Metabolic Modeling, Host–microbe Interaction, Metabolomics, Outer Membrane Vesicle</p>
                    <p>The interactions between the gut microbiota and its host are of central importance to the health of the host. Outer membrane vesicles (OMVs) are produced ubiquitously by Gram-negative bacteria including the gut commensal</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.3389/fmicb.2017.02440">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29276507">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Determinism and Contingency Shape Metabolic Complementation in an Endosymbiotic Consortium.</h1>
                    <h3 class="mdl-color-text--gray header">Frontiers in Microbiology&nbsp;2017</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Miguel Ponce-de-Leon, Daniel Tamarit, Jorge Calle-Espinosa, Matteo Mori, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Cross-Feeding, Endosymbiotic Bacteria, Metabolic Evolution, Metabolic Modeling, Stoichiometric Analysis</p>
                    <p>Bacterial endosymbionts and their insect hosts establish an intimate metabolic relationship. Bacteria offer a variety of essential nutrients to their hosts, whereas insect cells provide the necessary sources of matter and energy to their tiny metabolic allies. These nutritional complementations sustain themselves on a diversity of metabolite exchanges between the cell host and the reduced yet highly specialized bacterial metabolism-which, for instance, overproduces a small set of essential amino acids and vitamins.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.3389/fmicb.2017.02290">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29213256">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Characterizing steady states of genome-scale metabolic networks in continuous cell cultures.</h1>
                    <h3 class="mdl-color-text--gray header">PLoS Computational Biology&nbsp;2017</h3>
                    
                    <h2 class="mdl-color-text--primary header">Jorge Fernandez-de-Cossio-Diaz, Kalet Leon and Roberto Mulet</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>In the continuous mode of cell culture, a constant flow carrying fresh media replaces culture fluid, cells, nutrients and secreted metabolites. Here we present a model for continuous cell culture coupling intra-cellular metabolism to extracellular variables describing the state of the bioreactor, taking into account the growth capacity of the cell and the impact of toxic byproduct accumulation. We provide a method to determine the steady states of this system that is tractable for metabolic networks of arbitrary complexity.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pcbi.1005835">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29131817">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Oxygen-limited metabolism in the methanotroph <i>Methylomicrobium buryatense</i> 5GB1C.</h1>
                    <h3 class="mdl-color-text--gray header">PeerJ&nbsp;2017</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Alexey Gilman, Yanfen Fu, Melissa Hendershott, Frances Chu, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Acetate, Excretion Products, Methane, Methanotroph</p>
                    <p>The bacteria that grow on methane aerobically (methanotrophs) support populations of non-methanotrophs in the natural environment by excreting methane-derived carbon. One group of excreted compounds are short-chain organic acids, generated in highest abundance when cultures are grown under O</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.7717/peerj.3945">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29062611">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Combining graph and flux-based structures to decipher phenotypic essential metabolites within metabolic networks.</h1>
                    <h3 class="mdl-color-text--gray header">PeerJ&nbsp;2017</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Julie Laniau, Clémence Frioux, Jacques Nicolas, Caroline Baroukh, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Answer Set Programming, Constraint-Based Analysis, Essential Metabolite, Graph-Based Analysis, Metabolic Networks</p>
                    <p>The emergence of functions in biological systems is a long-standing issue that can now be addressed at the cell level with the emergence of high throughput technologies for genome sequencing and phenotyping. The reconstruction of complete metabolic networks for various organisms is a key outcome of the analysis of these data, giving access to a global view of cell functioning. The analysis of metabolic networks may be carried out by simply considering the architecture of the reaction network or by taking into account the stoichiometry of reactions.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.7717/peerj.3860">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29038751">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Genome-Scale Architecture of Small Molecule Regulatory Networks and the Fundamental Trade-Off between Regulation and Enzymatic Activity.</h1>
                    <h3 class="mdl-color-text--gray header">Cell Reports&nbsp;2017</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Ed Reznik, Dimitris Christodoulou, Joshua E Goldford, Emma Briars, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Elasticity, Enzyme Kinetics, Metabolism, Regulatory Network, Small Molecule Regulation, Trade-Offs</p>
                    <p>Metabolic flux is in part regulated by endogenous small molecules that modulate the catalytic activity of an enzyme, e.g., allosteric inhibition. In contrast to transcriptional regulation of enzymes, technical limitations have hindered the production of a genome-scale atlas of small molecule-enzyme regulatory interactions. Here, we develop a framework leveraging the vast, but fragmented, biochemical literature to reconstruct and analyze the small molecule regulatory network (SMRN) of the model organism Escherichia coli, including the primary metabolite regulators and enzyme targets.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1016/j.celrep.2017.08.066">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/28903046">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Population FBA predicts metabolic phenotypes in yeast.</h1>
                    <h3 class="mdl-color-text--gray header">PLoS Computational Biology&nbsp;2017</h3>
                    
                    <h2 class="mdl-color-text--primary header">Piyush Labhsetwar, Marcelo C R Melo, John A Cole and Zaida Luthey-Schulten</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Using protein counts sampled from single cell proteomics distributions to constrain fluxes through a genome-scale model of metabolism, Population flux balance analysis (Population FBA) successfully described metabolic heterogeneity in a population of independent Escherichia coli cells growing in a defined medium. We extend the methodology to account for correlations in protein expression arising from the co-regulation of genes and apply it to study the growth of independent Saccharomyces cerevisiae cells in two different growth media.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pcbi.1005728">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/28886026">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Parametric studies of metabolic cooperativity in Escherichia coli colonies: Strain and geometric confinement effects.</h1>
                    <h3 class="mdl-color-text--gray header">PloS One&nbsp;2017</h3>
                    
                    <h2 class="mdl-color-text--primary header">Joseph R Peterson, John A Cole and Zaida Luthey-Schulten</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Characterizing the complex spatial and temporal interactions among cells in a biological system (i.e. bacterial colony, microbiome, tissue, etc.) remains a challenge. Metabolic cooperativity in these systems can arise due to the subtle interplay between microenvironmental conditions and the cells&rsquo; regulatory machinery, often involving cascades of intra- and extracellular signalling molecules. In the simplest of cases, as demonstrated in a recent study of the model organism Escherichia coli, metabolic cross-feeding can arise in monoclonal colonies of bacteria driven merely by spatial heterogeneity in the availability of growth substrates; namely, acetate, glucose and oxygen.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pone.0182570">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/28820904">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Metabolic modeling of energy balances in Mycoplasma hyopneumoniae shows that pyruvate addition increases growth rate.</h1>
                    <h3 class="mdl-color-text--gray header">Biotechnology and Bioengineering&nbsp;2017</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Tjerko Kamminga, Simen-Jan Slagman, Jetta J E Bijlsma, Vitor A P Martins Dos Santos, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Mycoplasma Hyopneumoniae, Constraint-Based Metabolic Modeling, Energy Balances, Metabolic Networks, Process Optimization</p>
                    <p>Mycoplasma hyopneumoniae is cultured on large-scale to produce antigen for inactivated whole-cell vaccines against respiratory disease in pigs. However, the fastidious nutrient requirements of this minimal bacterium and the low growth rate make it challenging to reach sufficient biomass yield for antigen production. In this study, we sequenced the genome of M. hyopneumoniae strain 11 and constructed a high quality constraint-based genome-scale metabolic model of 284 chemical reactions and 298 metabolites.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1002/bit.26347">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/28600895">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Constraint-based modeling identifies new putative targets to fight colistin-resistant A. baumannii infections.</h1>
                    <h3 class="mdl-color-text--gray header">Scientific Reports&nbsp;2017</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Luana Presta, Emanuele Bosi, Leila Mansouri, Lenie Dijkshoorn, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Acinetobacter baumannii is a clinical threat to human health, causing major infection outbreaks worldwide. As new drugs against Gram-negative bacteria do not seem to be forthcoming, and due to the microbial capability of acquiring multi-resistance, there is an urgent need for novel therapeutic targets. Here we have derived a list of new potential targets by means of metabolic reconstruction and modelling of A. baumannii ATCC 19606. By integrating constraint-based modelling with gene expression data, we simulated microbial growth in normal and stressful conditions (i.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1038/s41598-017-03416-2">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/28623298">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Combining Genome-Scale Experimental and Computational Methods To Identify Essential Genes in <i>Rhodobacter sphaeroides</i>.</h1>
                    <h3 class="mdl-color-text--gray header">MSystems&nbsp;2017</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Brian T Burger, Saheed Imam, Matthew J Scarborough, Daniel R Noguera, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Rhodobacter Sphaeroides, Tn-Seq, Gene Disruption, Genomics, Metabolic Modeling, Metabolism, Photosynthetic Bacteria, Proteobacteria</p>
                    <p>None</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1128/mSystems.00015-17">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/28744485">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">DistributedFBA.jl: high-level, high-performance flux balance analysis in Julia.</h1>
                    <h3 class="mdl-color-text--gray header">Bioinformatics (Oxford, England)&nbsp;2017</h3>
                    
                    <h2 class="mdl-color-text--primary header">Laurent Heirendt, Ines Thiele and Ronan M T Fleming</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Flux balance analysis and its variants are widely used methods for predicting steady-state reaction rates in biochemical reaction networks. The exploration of high dimensional networks with such methods is currently hampered by software performance limitations.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1093/bioinformatics/btw838">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/28453682">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Mackinac: a bridge between ModelSEED and COBRApy to generate and analyze genome-scale metabolic models.</h1>
                    <h3 class="mdl-color-text--gray header">Bioinformatics (Oxford, England)&nbsp;2017</h3>
                    
                    <h2 class="mdl-color-text--primary header">Michael Mundy, Helena Mendes-Soares and Nicholas Chia</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Reconstructing and analyzing a large number of genome-scale metabolic models is a fundamental part of the integrated study of microbial communities; however, two of the most widely used frameworks for building and analyzing models use different metabolic network representations. Here we describe Mackinac, a Python package that combines ModelSEED&rsquo;s ability to automatically reconstruct metabolic models with COBRApy&rsquo;s advanced analysis capabilities to bridge the differences between the two frameworks and facilitate the study of the metabolic potential of microorganisms.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1093/bioinformatics/btx185">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/28379466">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">The JBEI quantitative metabolic modeling library (jQMM): a python library for modeling microbial metabolism.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Bioinformatics&nbsp;2017</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Garrett W Birkel, Amit Ghosh, Vinay S Kumar, Daniel Weaver, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        -Omics Data, 13 C Metabolic Flux Analysis, Flux Analysis, Predictive Biology</p>
                    <p>Modeling of microbial metabolism is a topic of growing importance in biotechnology. Mathematical modeling helps provide a mechanistic understanding for the studied process, separating the main drivers from the circumstantial ones, bounding the outcomes of experiments and guiding engineering approaches. Among different modeling schemes, the quantification of intracellular metabolic fluxes (i.e. the rate of each reaction in cellular metabolism) is of particular interest for metabolic engineering because it describes how carbon and energy flow throughout the cell.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s12859-017-1615-y">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/28381205">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Systems approach to characterize the metabolism of liver cancer stem cells expressing CD133.</h1>
                    <h3 class="mdl-color-text--gray header">Scientific Reports&nbsp;2017</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Wonhee Hur, Jae Yong Ryu, Hyun Uk Kim, Sung Woo Hong, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Liver cancer stem cells (LCSCs) have attracted attention because they cause therapeutic resistance in hepatocellular carcinoma (HCC). Understanding the metabolism of LCSCs can be a key to developing therapeutic strategy, but metabolic characteristics have not yet been studied. Here, we systematically analyzed and compared the global metabolic phenotype between LCSCs and non-LCSCs using transcriptome and metabolome data. We also reconstructed genome-scale metabolic models (GEMs) for LCSC and non-LCSC to comparatively examine differences in their metabolism at genome-scale.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1038/srep45557">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/28367990">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Efficient estimation of the maximum metabolic productivity of batch systems.</h1>
                    <h3 class="mdl-color-text--gray header">Biotechnology for Biofuels&nbsp;2017</h3>
                    
                    <h2 class="mdl-color-text--primary header">Peter C St John, Michael F Crowley and Yannick J Bomble</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Actinobacillus Succinogenes, Dynamic Optimizations, Elementary Flux Modes, Escherichia Coli, Flux Balance Analysis</p>
                    <p>Production of chemicals from engineered organisms in a batch culture involves an inherent trade-off between productivity, yield, and titer. Existing strategies for strain design typically focus on designing mutations that achieve the highest yield possible while maintaining growth viability. While these methods are computationally tractable, an optimum productivity could be achieved by a dynamic strategy in which the intracellular division of resources is permitted to change with time. New methods for the design and implementation of dynamic microbial processes, both computational and experimental, have therefore been explored to maximize productivity.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s13068-017-0709-0">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/28163785">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Meneco, a Topology-Based Gap-Filling Tool Applicable to Degraded Genome-Wide Metabolic Networks.</h1>
                    <h3 class="mdl-color-text--gray header">PLoS Computational Biology&nbsp;2017</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Sylvain Prigent, Clémence Frioux, Simon M Dittami, Sven Thiele, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Increasing amounts of sequence data are becoming available for a wide range of non-model organisms. Investigating and modelling the metabolic behaviour of those organisms is highly relevant to understand their biology and ecology. As sequences are often incomplete and poorly annotated, draft networks of their metabolism largely suffer from incompleteness. Appropriate gap-filling methods to identify and add missing reactions are therefore required to address this issue. However, current tools rely on phenotypic or taxonomic information, or are very sensitive to the stoichiometric balance of metabolic reactions, especially concerning the co-factors.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pcbi.1005276">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/28129330">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Reliable and efficient solution of genome-scale models of Metabolism and macromolecular Expression.</h1>
                    <h3 class="mdl-color-text--gray header">Scientific Reports&nbsp;2017</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Ding Ma, Laurence Yang, Ronan M T Fleming, Ines Thiele, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Constraint-Based Reconstruction and Analysis (COBRA) is currently the only methodology that permits integrated modeling of Metabolism and macromolecular Expression (ME) at genome-scale. Linear optimization computes steady-state flux solutions to ME models, but flux values are spread over many orders of magnitude. Data values also have greatly varying magnitudes. Standard double-precision solvers may return inaccurate solutions or report that no solution exists. Exact simplex solvers based on rational arithmetic require a near-optimal warm start to be practical on large problems (current ME models have 70,000 constraints and variables and will grow larger).</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1038/srep40863">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/28098205">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Genome-Scale Metabolic Modeling of Archaea Lends Insight into Diversity of Metabolic Function.</h1>
                    <h3 class="mdl-color-text--gray header">Archaea (Vancouver, B.C.)&nbsp;2017</h3>
                    
                    <h2 class="mdl-color-text--primary header">ShengShee Thor, Joseph R Peterson and Zaida Luthey-Schulten</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Decades of biochemical, bioinformatic, and sequencing data are currently being systematically compiled into genome-scale metabolic reconstructions (GEMs). Such reconstructions are knowledge-bases useful for engineering, modeling, and comparative analysis. Here we review the fifteen GEMs of archaeal species that have been constructed to date. They represent primarily members of the Euryarchaeota with three-quarters comprising representative of methanogens. Unlike other reviews on GEMs, we specially focus on archaea. We briefly review the GEM construction process and the genealogy of the archaeal models.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1155/2017/9763848">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/28133437">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <h6>Total of 21 publications. 😄</h6>
        </div>
    </div>
    
    <div class="mdl-tabs__panel " id="year_2016">
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Personalized Prediction of Proliferation Rates and Metabolic Liabilities in Cancer Biopsies.</h1>
                    <h3 class="mdl-color-text--gray header">Frontiers in Physiology&nbsp;2016</h3>
                    
                    <h2 class="mdl-color-text--primary header">Christian Diener and Osbaldo Resendis-Antonio</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        NCI60, TCGA, Flux Balance Analysis, Personalized Medicine, Proliferation, Systems Biology</p>
                    <p>Cancer is a heterogeneous disease and its genetic and metabolic mechanism may manifest differently in each patient. This creates a demand for studies that can characterize phenotypic traits of cancer on a per-sample basis. Combining two large data sets, the NCI60 cancer cell line panel, and The Cancer Genome Atlas, we used a linear interaction model to predict proliferation rates for more than 12,000 cancer samples across 33 different cancers from The Cancer Genome Atlas.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.3389/fphys.2016.00644">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/28082911">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Community metabolic modeling approaches to understanding the gut microbiome: Bridging biochemistry and ecology.</h1>
                    <h3 class="mdl-color-text--gray header">Free Radical Biology &amp; Medicine&nbsp;2016</h3>
                    
                    <h2 class="mdl-color-text--primary header">Helena Mendes-Soares and Nicholas Chia</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Community Metabolic Models, Ecological Models, Metabolic Models, Microbiome</p>
                    <p>Interest in the human microbiome is at an all time high. The number of human microbiome studies is growing exponentially, as are reported associations between microbial communities and disease. However, we have not been able to translate the ever-growing amount of microbiome sequence data into better health. To do this, we need a practical means of transforming a disease-associated microbiome into a health-associated microbiome. This will require a framework that can be used to generate predictions about community dynamics within the microbiome under different conditions, predictions that can be tested and validated.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1016/j.freeradbiomed.2016.12.017">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/27989793">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Unique attributes of cyanobacterial metabolism revealed by improved genome-scale metabolic modeling and essential gene analysis.</h1>
                    <h3 class="mdl-color-text--gray header">Proceedings of the National Academy of Sciences of the United States of America&nbsp;2016</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Jared T Broddrick, Benjamin E Rubin, David G Welkie, Niu Du, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Synechococcus Elongatus, TCA Cycle, Constraint-Based Modeling, Cyanobacteria, Photosynthesis</p>
                    <p>The model cyanobacterium, Synechococcus elongatus PCC 7942, is a genetically tractable obligate phototroph that is being developed for the bioproduction of high-value chemicals. Genome-scale models (GEMs) have been successfully used to assess and engineer cellular metabolism; however, GEMs of phototrophic metabolism have been limited by the lack of experimental datasets for model validation and the challenges of incorporating photon uptake. Here, we develop a GEM of metabolism in S. elongatus using random barcode transposon site sequencing (RB-TnSeq) essential gene and physiological data specific to photoautotrophic metabolism.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1073/pnas.1613446113">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/27911809">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Modelling microbial metabolic rewiring during growth in a complex medium.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Genomics&nbsp;2016</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Marco Fondi, Emanuele Bosi, Luana Presta, Diletta Natoli, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Antarctic Bacteria, Flux Balance Analysis, Metabolic Modelling, Pseudoalteromonas Haloplanktis TAC125</p>
                    <p>In their natural environment, bacteria face a wide range of environmental conditions that change over time and that impose continuous rearrangements at all the cellular levels (e.g. gene expression, metabolism). When facing a nutritionally rich environment, for example, microbes first use the preferred compound(s) and only later start metabolizing the other one(s). A systemic re-organization of the overall microbial metabolic network in response to a variation in the composition/concentration of the surrounding nutrients has been suggested, although the range and the entity of such modifications in organisms other than a few model microbes has been scarcely described up to now.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s12864-016-3311-0">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/27881075">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Multi-omic data integration enables discovery of hidden biological regularities.</h1>
                    <h3 class="mdl-color-text--gray header">Nature Communications&nbsp;2016</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Ali Ebrahim, Elizabeth Brunk, Justin Tan, Edward J O&#39;Brien, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Rapid growth in size and complexity of biological data sets has led to the &lsquo;Big Data to Knowledge&rsquo; challenge. We develop advanced data integration methods for multi-level analysis of genomic, transcriptomic, ribosomal profiling, proteomic and fluxomic data. First, we show that pairwise integration of primary omics data reveals regularities that tie cellular processes together in Escherichia coli: the number of protein molecules made per mRNA transcript and the number of ribosomes required per translated protein molecule.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1038/ncomms13091">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/27782110">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">solveME: fast and reliable solution of nonlinear ME models.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Bioinformatics&nbsp;2016</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Laurence Yang, Ding Ma, Ali Ebrahim, Colton J Lloyd, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Constraint-Based Modeling, Metabolism, Nonlinear Optimization, Proteome, Quasiconvex</p>
                    <p>Genome-scale models of metabolism and macromolecular expression (ME) significantly expand the scope and predictive capabilities of constraint-based modeling. ME models present considerable computational challenges: they are much (&gt;30 times) larger than corresponding metabolic reconstructions (M models), are multiscale, and growth maximization is a nonlinear programming (NLP) problem, mainly due to macromolecule dilution constraints.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s12859-016-1240-1">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/27659412">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Multi-omics Quantification of Species Variation of Escherichia coli Links Molecular Features with Strain Phenotypes.</h1>
                    <h3 class="mdl-color-text--gray header">Cell Systems&nbsp;2016</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Jonathan M Monk, Anna Koza, Miguel A Campodonico, Daniel Machado, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Escherichia Coli, Genome-Scale Modeling, Metabolic Engineering, Systems Biology</p>
                    <p>Escherichia coli strains are widely used in academic research and biotechnology. New technologies for quantifying strain-specific differences and their underlying contributing factors promise greater understanding of how these differences significantly impact physiology, synthetic biology, metabolic engineering, and process design. Here, we quantified strain-specific differences in seven widely used strains of E. coli (BL21, C, Crooks, DH5a, K-12 MG1655, K-12 W3110, and W) using genomics, phenomics, transcriptomics, and genome-scale modeling. Metabolic physiology and gene expression varied widely with downstream implications for productivity, product yield, and titer.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1016/j.cels.2016.08.013">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/27667363">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">MMinte: an application for predicting metabolic interactions among the microbial species in a community.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Bioinformatics&nbsp;2016</h3>
                    
                    <h2 class="mdl-color-text--primary header">Helena Mendes-Soares, Michael Mundy, Luis Mendes Soares and Nicholas Chia</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        16S RDNA, Metabolic Network Reconstruction, Microbiome, Network, Predictive Community Modeling</p>
                    <p>The explosive growth of microbiome research has yielded great quantities of data. These data provide us with many answers, but raise just as many questions. 16S rDNA-the backbone of microbiome analyses-allows us to assess α-diversity, β-diversity, and microbe-microbe associations, which characterize the overall properties of an ecosystem. However, we are still unable to use 16S rDNA data to directly assess the microbe-microbe and microbe-environment interactions that determine the broader ecology of that system.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s12859-016-1230-3">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/27590448">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Quantifying complexity in metabolic engineering using the LASER database.</h1>
                    <h3 class="mdl-color-text--gray header">Metabolic Engineering Communications&nbsp;2016</h3>
                    
                    <h2 class="mdl-color-text--primary header">James D Winkler, Andrea L Halweg-Edwards and Ryan T Gill</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Design Tools, Metabolic Engineering, Standardization, Synthetic Biology</p>
                    <p>We previously introduced the LASER database (Learning Assisted Strain EngineeRing, https://bitbucket.org/jdwinkler/laser_release) (Winkler et al. 2015) to serve as a platform for understanding past and present metabolic engineering practices. Over the past year, LASER has been expanded by 50% to include over 600 engineered strains from 450 papers, including their growth conditions, genetic modifications, and other information in an easily searchable format. Here, we present the results of our efforts to use LASER as a means for defining the complexity of a metabolic engineering &ldquo;design&rdquo;.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1016/j.meteno.2016.07.002">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/29468127">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Genome-Scale Metabolic Model for the Green Alga Chlorella vulgaris UTEX 395 Accurately Predicts Phenotypes under Autotrophic, Heterotrophic, and Mixotrophic Growth Conditions.</h1>
                    <h3 class="mdl-color-text--gray header">Plant Physiology&nbsp;2016</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Cristal Zuñiga, Chien-Ting Li, Tyler Huelsman, Jennifer Levering, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>The green microalga Chlorella vulgaris has been widely recognized as a promising candidate for biofuel production due to its ability to store high lipid content and its natural metabolic versatility. Compartmentalized genome-scale metabolic models constructed from genome sequences enable quantitative insight into the transport and metabolism of compounds within a target organism. These metabolic models have long been utilized to generate optimized design strategies for an improved production process. Here, we describe the reconstruction, validation, and application of a genome-scale metabolic model for C.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1104/pp.16.00593">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/27372244">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Integrated In Silico Analysis of Pathway Designs for Synthetic Photo-Electro-Autotrophy.</h1>
                    <h3 class="mdl-color-text--gray header">PloS One&nbsp;2016</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Michael Volpers, Nico J Claassens, Elad Noor, John van der Oost, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>The strong advances in synthetic biology enable the engineering of novel functions and complex biological features in unprecedented ways, such as implementing synthetic autotrophic metabolism into heterotrophic hosts. A key challenge for the sustainable production of fuels and chemicals entails the engineering of synthetic autotrophic organisms that can effectively and efficiently fix carbon dioxide by using sustainable energy sources. This challenge involves the integration of carbon fixation and energy uptake systems.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pone.0157851">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/27336167">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">From DNA to FBA: How to Build Your Own Genome-Scale Metabolic Model.</h1>
                    <h3 class="mdl-color-text--gray header">Frontiers in Microbiology&nbsp;2016</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Daniel A Cuevas, Janaka Edirisinghe, Chris S Henry, Ross Overbeek, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Flux-Balance Analysis, Genome Annotation, in Silico Modeling, Metabolic Modeling, Metabolic Reconstruction, Model SEED</p>
                    <p>Microbiological studies are increasingly relying on in silico methods to perform exploration and rapid analysis of genomic data, and functional genomics studies are supplemented by the new perspectives that genome-scale metabolic models offer. A mathematical model consisting of a microbe&rsquo;s entire metabolic map can be rapidly determined from whole-genome sequencing and annotating the genomic material encoded in its DNA. Flux-balance analysis (FBA), a linear programming technique that uses metabolic models to predict the phenotypic responses imposed by environmental elements and factors, is the leading method to simulate and manipulate cellular growth in silico.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.3389/fmicb.2016.00907">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/27379044">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Toward Community Standards and Software for Whole-Cell Modeling.</h1>
                    <h3 class="mdl-color-text--gray header">IEEE Transactions on Bio-Medical Engineering&nbsp;2016</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Dagmar Waltemath, Jonathan R Karr, Frank T Bergmann, Vijayalakshmi Chelliah, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Whole-cell (WC) modeling is a promising tool for biological research, bioengineering, and medicine. However, substantial work remains to create accurate comprehensive models of complex cells.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1109/TBME.2016.2560762">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/27305665">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Comparative genome-scale modelling of Staphylococcus aureus strains identifies strain-specific metabolic capabilities linked to pathogenicity.</h1>
                    <h3 class="mdl-color-text--gray header">Proceedings of the National Academy of Sciences of the United States of America&nbsp;2016</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Emanuele Bosi, Jonathan M Monk, Ramy K Aziz, Marco Fondi, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Core Genome, Mathematical Modeling, Pangenome, Pathogenicity, Systems Biology</p>
                    <p>Staphylococcus aureus is a preeminent bacterial pathogen capable of colonizing diverse ecological niches within its human host. We describe here the pangenome of S. aureus based on analysis of genome sequences from 64 strains of S. aureus spanning a range of ecological niches, host types, and antibiotic resistance profiles. Based on this set, S. aureus is expected to have an open pangenome composed of 7,411 genes and a core genome composed of 1,441 genes.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1073/pnas.1523199113">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/27286824">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Systems biology of the structural proteome.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Systems Biology&nbsp;2016</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Elizabeth Brunk, Nathan Mih, Jonathan Monk, Zhen Zhang, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>The success of genome-scale models (GEMs) can be attributed to the high-quality, bottom-up reconstructions of metabolic, protein synthesis, and transcriptional regulatory networks on an organism-specific basis. Such reconstructions are biochemically, genetically, and genomically structured knowledge bases that can be converted into a mathematical format to enable a myriad of computational biological studies. In recent years, genome-scale reconstructions have been extended to include protein structural information, which has opened up new vistas in systems biology research and empowered applications in structural systems biology and systems pharmacology.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s12918-016-0271-6">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/26969117">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Global characterization of in vivo enzyme catalytic rates and their correspondence to in vitro kcat measurements.</h1>
                    <h3 class="mdl-color-text--gray header">Proceedings of the National Academy of Sciences of the United States of America&nbsp;2016</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Dan Davidi, Elad Noor, Wolfram Liebermeister, Arren Bar-Even, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Flux Balance Analysis, Kcat, Kinetic Constants, Proteomics, Turnover Number</p>
                    <p>Turnover numbers, also known as kcat values, are fundamental properties of enzymes. However, kcat data are scarce and measured in vitro, thus may not faithfully represent the in vivo situation. A basic question that awaits elucidation is: how representative are kcat values for the maximal catalytic rates of enzymes in vivo? Here, we harness omics data to calculate kmax(vivo), the observed maximal catalytic rate of an enzyme inside cells. Comparison with kcat values from Escherichia coli, yields a correlation ofr(2)= 0.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1073/pnas.1514240113">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/26951675">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">What Makes a Bacterial Species Pathogenic?:Comparative Genomic Analysis of the Genus Leptospira.</h1>
                    <h3 class="mdl-color-text--gray header">PLoS Neglected Tropical Diseases&nbsp;2016</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Derrick E Fouts, Michael A Matthias, Haritha Adhikarla, Ben Adler, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Leptospirosis, caused by spirochetes of the genus Leptospira, is a globally widespread, neglected and emerging zoonotic disease. While whole genome analysis of individual pathogenic, intermediately pathogenic and saprophytic Leptospira species has been reported, comprehensive cross-species genomic comparison of all known species of infectious and non-infectious Leptospira, with the goal of identifying genes related to pathogenesis and mammalian host adaptation, remains a key gap in the field. Infectious Leptospira, comprised of pathogenic and intermediately pathogenic Leptospira, evolutionarily diverged from non-infectious, saprophytic Leptospira, as demonstrated by the following computational biology analyses: 1) the definitive taxonomy and evolutionary relatedness among all known Leptospira species; 2) genomically-predicted metabolic reconstructions that indicate novel adaptation of infectious Leptospira to mammals, including sialic acid biosynthesis, pathogen-specific porphyrin metabolism and the first-time demonstration of cobalamin (B12) autotrophy as a bacterial virulence factor; 3) CRISPR/Cas systems demonstrated only to be present in pathogenic Leptospira, suggesting a potential mechanism for this clade&rsquo;s refractoriness to gene targeting; 4) finding Leptospira pathogen-specific specialized protein secretion systems; 5) novel virulence-related genes/gene families such as the Virulence Modifying (VM) (PF07598 paralogs) proteins and pathogen-specific adhesins; 6) discovery of novel, pathogen-specific protein modification and secretion mechanisms including unique lipoprotein signal peptide motifs, Sec-independent twin arginine protein secretion motifs, and the absence of certain canonical signal recognition particle proteins from all Leptospira; and 7) and demonstration of infectious Leptospira-specific signal-responsive gene expression, motility and chemotaxis systems.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pntd.0004403">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/26890609">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">PSAMM: A Portable System for the Analysis of Metabolic Models.</h1>
                    <h3 class="mdl-color-text--gray header">PLoS Computational Biology&nbsp;2016</h3>
                    
                    <h2 class="mdl-color-text--primary header">Jon Lund Steffensen, Keith Dufault-Thompson and Ying Zhang</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>The genome-scale models of metabolic networks have been broadly applied in phenotype prediction, evolutionary reconstruction, community functional analysis, and metabolic engineering. Despite the development of tools that support individual steps along the modeling procedure, it is still difficult to associate mathematical simulation results with the annotation and biological interpretation of metabolic models. In order to solve this problem, here we developed a Portable System for the Analysis of Metabolic Models (PSAMM), a new open-source software package that supports the integration of heterogeneous metadata in model annotations and provides a user-friendly interface for the analysis of metabolic models.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pcbi.1004732">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/26828591">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <h6>Total of 18 publications. 😄</h6>
        </div>
    </div>
    
    <div class="mdl-tabs__panel " id="year_2015">
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">FlexFlux: combining metabolic flux and regulatory network analyses.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Systems Biology&nbsp;2015</h3>
                    
                    <h2 class="mdl-color-text--primary header">Lucas Marmiesse, Rémi Peyraud and Ludovic Cottret</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Expression of cell phenotypes highly depends on metabolism that supplies matter and energy. To achieve proper utilisation of the different metabolic pathways, metabolism is tightly regulated by a complex regulatory network composed of diverse biological entities (genes, transcripts, proteins, signalling molecules…). The integrated analysis of both regulatory and metabolic networks appears very insightful but is not straightforward because of the distinct characteristics of both networks. The classical method used for metabolic flux analysis is Flux Balance Analysis (FBA), which is constraint-based and relies on the assumption of steady-state metabolite concentrations throughout the network.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/s12918-015-0238-z">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/26666757">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Consistency Analysis of Genome-Scale Models of Bacterial Metabolism: A Metamodel Approach.</h1>
                    <h3 class="mdl-color-text--gray header">PloS One&nbsp;2015</h3>
                    
                    <h2 class="mdl-color-text--primary header">Miguel Ponce-de-Leon, Jorge Calle-Espinosa, Juli Peretó and Francisco Montero</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Genome-scale metabolic models usually contain inconsistencies that manifest as blocked reactions and gap metabolites. With the purpose to detect recurrent inconsistencies in metabolic models, a large-scale analysis was performed using a previously published dataset of 130 genome-scale models. The results showed that a large number of reactions (~22%) are blocked in all the models where they are present. To unravel the nature of such inconsistencies a metamodel was construed by joining the 130 models in a single network.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pone.0143626">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/26629901">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Networks of energetic and metabolic interactions define dynamics in microbial communities.</h1>
                    <h3 class="mdl-color-text--gray header">Proceedings of the National Academy of Sciences of the United States of America&nbsp;2015</h3>
                    
                    <h2 class="mdl-color-text--primary header">Mallory Embree, Joanne K Liu, Mahmoud M Al-Bassam and Karsten Zengler</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Interspecies Interactions, Metabolic Modeling, Methanogens, Microbial Communities, Microbiome</p>
                    <p>Microorganisms form diverse communities that have a profound impact on the environment and human health. Recent technological advances have enabled elucidation of community diversity at high resolution. Investigation of microbial communities has revealed that they often contain multiple members with complementing and seemingly redundant metabolic capabilities. An understanding of the communal impacts of redundant metabolic capabilities is currently lacking; specifically, it is not known whether metabolic redundancy will foster competition or motivate cooperation.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1073/pnas.1506034112">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/26621749">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Systems biology-guided identification of synthetic lethal gene pairs and its potential use to discover antibiotic combinations.</h1>
                    <h3 class="mdl-color-text--gray header">Scientific Reports&nbsp;2015</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Ramy K Aziz, Jonathan M Monk, Robert M Lewis, Suh In Loh, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Mathematical models of metabolism from bacterial systems biology have proven their utility across multiple fields, for example metabolic engineering, growth phenotype simulation, and biological discovery. The usefulness of the models stems from their ability to compute a link between genotype and phenotype, but their ability to accurately simulate gene-gene interactions has not been investigated extensively. Here we assess how accurately a metabolic model for Escherichia coli computes one particular type of gene-gene interaction, synthetic lethality, and find that the accuracy rate is between 25% and 43%.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1038/srep16025">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/26531810">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">The MetaCyc database of metabolic pathways and enzymes and the BioCyc collection of pathway/genome databases.</h1>
                    <h3 class="mdl-color-text--gray header">Nucleic Acids Research&nbsp;2015</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Ron Caspi, Richard Billington, Luciana Ferrer, Hartmut Foerster, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>The MetaCyc database (MetaCyc.org) is a freely accessible comprehensive database describing metabolic pathways and enzymes from all domains of life. The majority of MetaCyc pathways are small-molecule metabolic pathways that have been experimentally determined. MetaCyc contains more than 2400 pathways derived from &gt;46,000 publications, and is the largest curated collection of metabolic pathways. BioCyc (BioCyc.org) is a collection of 5700 organism-specific Pathway/Genome Databases (PGDBs), each containing the full genome and predicted metabolic network of one organism, including metabolites, enzymes, reactions, metabolic pathways, predicted operons, transport systems, and pathway-hole fillers.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1093/nar/gkv1164">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/26527732">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">BiGG Models: A platform for integrating, standardizing and sharing genome-scale models.</h1>
                    <h3 class="mdl-color-text--gray header">Nucleic Acids Research&nbsp;2015</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Zachary A King, Justin Lu, Andreas Dräger, Philip Miller, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Genome-scale metabolic models are mathematically-structured knowledge bases that can be used to predict metabolic pathway usage and growth phenotypes. Furthermore, they can generate and test hypotheses when integrated with experimental data. To maximize the value of these models, centralized repositories of high-quality models must be established, models must adhere to established standards and model components must be linked to relevant databases. Tools for model visualization further enhance their utility. To meet these needs, we present BiGG Models (http://bigg.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1093/nar/gkv1049">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/26476456">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Cell scale host-pathogen modeling: another branch in the evolution of constraint-based methods.</h1>
                    <h3 class="mdl-color-text--gray header">Frontiers in Microbiology&nbsp;2015</h3>
                    
                    <h2 class="mdl-color-text--primary header">Neema Jamshidi and Anu Raghunathan</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Constraint-Based Model, Flux Balance Analysis, Host-Pathogen, Mathematical Models, Omics-Technologies, Optimization Methods, Salmonella Typhimurium, Tuberculosis</p>
                    <p>Constraint-based models have become popular methods for systems biology as they enable the integration of complex, disparate datasets in a biologically cohesive framework that also supports the description of biological processes in terms of basic physicochemical constraints and relationships. The scope, scale, and application of genome scale models have grown from single cell bacteria to multi-cellular interaction modeling; host-pathogen modeling represents one of these examples at the current horizon of constraint-based methods.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.3389/fmicb.2015.01032">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/26500611">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Model-driven discovery of synergistic inhibitors against E. coli and S. enterica serovar Typhimurium targeting a novel synthetic lethal pair, aldA and prpC.</h1>
                    <h3 class="mdl-color-text--gray header">Frontiers in Microbiology&nbsp;2015</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Ramy K Aziz, Valerie L Khaw, Jonathan M Monk, Elizabeth Brunk, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Antibiotic Development, Bacterial Metabolism, Drug Discovery, Metabolic Reconstruction, Model-Based Drug Target Discovery, Pathway Gap Filling, Synthetic Lethality, Systems Biology</p>
                    <p>Mathematical models of biochemical networks form a cornerstone of bacterial systems biology. Inconsistencies between simulation output and experimental data point to gaps in knowledge about the fundamental biology of the organism. One such inconsistency centers on the gene aldA in Escherichia coli: it is essential in a computational model of E. coli metabolism, but experimentally it is not. Here, we reconcile this disparity by providing evidence that aldA and prpC form a synthetic lethal pair, as the double knockout could only be created through complementation with a plasmid-borne copy of aldA.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.3389/fmicb.2015.00958">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/26441892">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Escher: A Web Application for Building, Sharing, and Embedding Data-Rich Visualizations of Biological Pathways.</h1>
                    <h3 class="mdl-color-text--gray header">PLoS Computational Biology&nbsp;2015</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Zachary A King, Andreas Dräger, Ali Ebrahim, Nikolaus Sonnenschein, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Escher is a web application for visualizing data on biological pathways. Three key features make Escher a uniquely effective tool for pathway visualization. First, users can rapidly design new pathway maps. Escher provides pathway suggestions based on user data and genome-scale models, so users can draw pathways in a semi-automated way. Second, users can visualize data related to genes or proteins on the associated reactions and pathways, using rules that define which enzymes catalyze each reaction.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pcbi.1004321">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/26313928">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Tools for visualization and analysis of molecular networks, pathways, and -omics data.</h1>
                    <h3 class="mdl-color-text--gray header">Advances and Applications in Bioinformatics and Chemistry : AABC&nbsp;2015</h3>
                    
                    <h2 class="mdl-color-text--primary header">Jose M Villaveces, Prasanna Koti and Bianca H Habermann</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Biological Networks, Genes, Organisms, Protein-Protein Interactions, Proteins, Reactions, Signaling</p>
                    <p>Biological pathways have become the standard way to represent the coordinated reactions and actions of a series of molecules in a cell. A series of interconnected pathways is referred to as a biological network, which denotes a more holistic view on the entanglement of cellular reactions. Biological pathways and networks are not only an appropriate approach to visualize molecular reactions. They have also become one leading method in -omics data analysis and visualization.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.2147/AABC.S63534">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/26082651">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Using Genome-scale Models to Predict Biological Capabilities.</h1>
                    <h3 class="mdl-color-text--gray header">Cell&nbsp;2015</h3>
                    
                    <h2 class="mdl-color-text--primary header">Edward J O'Brien, Jonathan M Monk and Bernhard O Palsson</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Constraint-based reconstruction and analysis (COBRA) methods at the genome scale have been under development since the first whole-genome sequences appeared in the mid-1990s. A few years ago, this approach began to demonstrate the ability to predict a range of cellular functions, including cellular growth capabilities on various substrates and the effect of gene knockouts at the genome scale. Thus, much interest has developed in understanding and applying these methods to areas such as metabolic engineering, antibiotic design, and organismal and enzyme evolution.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1016/j.cell.2015.05.019">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/26000478">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Using bioconductor package BiGGR for metabolic flux estimation based on gene expression changes in brain.</h1>
                    <h3 class="mdl-color-text--gray header">PloS One&nbsp;2015</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Anand K Gavai, Farahaniza Supandi, Hannes Hettling, Paul Murrell, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Predicting the distribution of metabolic fluxes in biochemical networks is of major interest in systems biology. Several databases provide metabolic reconstructions for different organisms. Software to analyze flux distributions exists, among others for the proprietary MATLAB environment. Given the large user community for the R computing environment, a simple implementation of flux analysis in R appears desirable and will facilitate easy interaction with computational tools to handle gene expression data. We extended the R software package BiGGR, an implementation of metabolic flux analysis in R.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1371/journal.pone.0119016">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/25806817">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <h6>Total of 12 publications. 😄</h6>
        </div>
    </div>
    
    <div class="mdl-tabs__panel " id="year_2014">
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Integration of a constraint-based metabolic model of Brassica napus developing seeds with (13)C-metabolic flux analysis.</h1>
                    <h3 class="mdl-color-text--gray header">Frontiers in Plant Science&nbsp;2014</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Jordan O Hay, Hai Shi, Nicolas Heinzel, Inga Hebbelmann, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        13C-Metabolic Flux Analysis, Carbon Partitioning, Central Metabolism, Constraint-Based Reconstruction and Analysis, Loopless Flux Balance Analysis</p>
                    <p>The use of large-scale or genome-scale metabolic reconstructions for modeling and simulation of plant metabolism and integration of those models with large-scale omics and experimental flux data is becoming increasingly important in plant metabolic research. Here we report an updated version of bna572, a bottom-up reconstruction of oilseed rape (Brassica napus L.; Brassicaceae) developing seeds with emphasis on representation of biomass-component biosynthesis. New features include additional seed-relevant pathways for isoprenoid, sterol, phenylpropanoid, flavonoid, and choline biosynthesis.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.3389/fpls.2014.00724">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/25566296">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Synthetic biology outside the cell: linking computational tools to cell-free systems.</h1>
                    <h3 class="mdl-color-text--gray header">Frontiers in Bioengineering and Biotechnology&nbsp;2014</h3>
                    
                    <h2 class="mdl-color-text--primary header">Daniel D Lewis, Fernando D Villarreal, Fan Wu and Cheemeng Tan</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Artificial Cells, Cell-Free Systems, Computational Modeling, Deterministic and Stochastic Simulations, in Vitro Model, Predictive Modeling, Synthetic Biology</p>
                    <p>As mathematical models become more commonly integrated into the study of biology, a common language for describing biological processes is manifesting. Many tools have emerged for the simulation of in vivo synthetic biological systems, with only a few examples of prominent work done on predicting the dynamics of cell-free synthetic systems. At the same time, experimental biologists have begun to study dynamics of in vitro systems encapsulated by amphiphilic molecules, opening the door for the development of a new generation of biomimetic systems.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.3389/fbioe.2014.00066">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/25538941">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Improving collaboration by standardization efforts in systems biology.</h1>
                    <h3 class="mdl-color-text--gray header">Frontiers in Bioengineering and Biotechnology&nbsp;2014</h3>
                    
                    <h2 class="mdl-color-text--primary header">Andreas Dräger and Bernhard Ø Palsson</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Model Databases, Model Formats, Modeling Guidelines, Network Visualization, Ontologies, Software Support</p>
                    <p>Collaborative genome-scale reconstruction endeavors of metabolic networks would not be possible without a common, standardized formal representation of these systems. The ability to precisely define biological building blocks together with their dynamic behavior has even been considered a prerequisite for upcoming synthetic biology approaches. Driven by the requirements of such ambitious research goals, standardization itself has become an active field of research on nearly all levels of granularity in biology. In addition to the originally envisaged exchange of computational models and tool interoperability, new standards have been suggested for an unambiguous graphical display of biological phenomena, to annotate, archive, as well as to rank models, and to describe execution and the outcomes of simulation experiments.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.3389/fbioe.2014.00061">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/25538939">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Current advances in systems and integrative biology.</h1>
                    <h3 class="mdl-color-text--gray header">Computational and Structural Biotechnology Journal&nbsp;2014</h3>
                    
                    <h2 class="mdl-color-text--primary header">Scott W Robinson, Marco Fernandes and Holger Husi</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Computational Biology, Data Integration, Pathway Mapping, Systems Biology</p>
                    <p>Systems biology has gained a tremendous amount of interest in the last few years. This is partly due to the realization that traditional approaches focusing only on a few molecules at a time cannot describe the impact of aberrant or modulated molecular environments across a whole system. Furthermore, a hypothesis-driven study aims to prove or disprove its postulations, whereas a hypothesis-free systems approach can yield an unbiased and novel testable hypothesis as an end-result.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1016/j.csbj.2014.08.007">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/25379142">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Whole Cell Modeling: From Single Cells to Colonies.</h1>
                    <h3 class="mdl-color-text--gray header">Israel Journal of Chemistry&nbsp;2014</h3>
                    
                    <h2 class="mdl-color-text--primary header">John A Cole and Zaida Luthey-Schulten</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em">Keywords:
                        Colony Dynamics, Flux Balance Analysis, Kinetics, Metabolism, Stochastic Modeling</p>
                    <p>A great deal of research over the last several years has focused on how the inherent randomness in movements and reactivity of biomolecules can give rise to unexpected large-scale differences in the behavior of otherwise identical cells. Our own research has approached this problem from two vantage points - a microscopic kinetic view of the individual molecules (nucleic acids, proteins, etc.) diffusing and interacting in a crowded cellular environment; and a broader systems-level view of how enzyme variability can give rise to well-defined metabolic phenotypes.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1002/ijch.201300147">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/26989262">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">A genome-scale metabolic flux model of Escherichia coli K-12 derived from the EcoCyc database.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Systems Biology&nbsp;2014</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Daniel S Weaver, Ingrid M Keseler, Amanda Mackie, Ian T Paulsen, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Constraint-based models of Escherichia coli metabolic flux have played a key role in computational studies of cellular metabolism at the genome scale. We sought to develop a next-generation constraint-based E. coli model that achieved improved phenotypic prediction accuracy while being frequently updated and easy to use. We also sought to compare model predictions with experimental data to highlight open questions in E. coli biology.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/1752-0509-8-79">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/24974895">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">A data integration and visualization resource for the metabolic network of Synechocystis sp. PCC 6803.</h1>
                    <h3 class="mdl-color-text--gray header">Plant Physiology&nbsp;2014</h3>
                    
                    <h2 class="mdl-color-text--primary header">Timo R Maarleveld, Joost Boele, Frank J Bruggeman and Bas Teusink</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Data integration is a central activity in systems biology. The integration of genomic, transcript, protein, metabolite, flux, and computational data yields unprecedented information about the system level functioning of organisms. Often, data integration is done purely computationally, leaving the user with little insight in addition to statistical information. In this article, we present a visualization tool for the metabolic network of Synechocystis sp. PCC 6803, an important model cyanobacterium for sustainable biofuel production.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1104/pp.113.224394">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/24402049">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <h6>Total of 7 publications. 😄</h6>
        </div>
    </div>
    
    <div class="mdl-tabs__panel " id="year_2013">
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Sybil--efficient constraint-based modelling in R.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Systems Biology&nbsp;2013</h3>
                    
                    <h2 class="mdl-color-text--primary header">Gabriel Gelius-Dietrich, Abdelmoneim Amer Desouki, Claus Jonathan Fritzemeier and Martin J Lercher</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Constraint-based analyses of metabolic networks are widely used to simulate the properties of genome-scale metabolic networks. Publicly available implementations tend to be slow, impeding large scale analyses such as the genome-wide computation of pairwise gene knock-outs, or the automated search for model improvements. Furthermore, available implementations cannot easily be extended or adapted by users.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/1752-0509-7-125">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/24224957">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Path2Models: large-scale generation of computational models from biochemical pathway maps.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Systems Biology&nbsp;2013</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Finja Büchel, Nicolas Rodriguez, Neil Swainston, Clemens Wrzodek, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Systems biology projects and omics technologies have led to a growing number of biochemical pathway models and reconstructions. However, the majority of these models are still created de novo, based on literature mining and the manual processing of pathway data.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/1752-0509-7-116">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/24180668">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">Solving gap metabolites and blocked reactions in genome-scale models: application to the metabolic network of Blattabacterium cuenoti.</h1>
                    <h3 class="mdl-color-text--gray header">BMC Systems Biology&nbsp;2013</h3>
                    
                    <h2 class="mdl-color-text--primary header">Miguel Ponce-de-León, Francisco Montero and Juli Peretó</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Metabolic reconstruction is the computational-based process that aims to elucidate the network of metabolites interconnected through reactions catalyzed by activities assigned to one or more genes. Reconstructed models may contain inconsistencies that appear as gap metabolites and blocked reactions. Although automatic methods for solving this problem have been previously developed, there are many situations where manual curation is still needed.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1186/1752-0509-7-114">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/24176055">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <div class="mdl-card mdl-shadow--4dp">
                <div class="mdl-card__supporting-text">
                    <h1 class="mdl-color-text--accent header">GIM3E: condition-specific models of cellular metabolism developed from metabolomics and expression data.</h1>
                    <h3 class="mdl-color-text--gray header">Bioinformatics (Oxford, England)&nbsp;2013</h3>
                    
                    
                    <h2 class="mdl-color-text--primary header">Brian J Schmidt, Ali Ebrahim, Thomas O Metz, Joshua N Adkins, et. al.</h2>
                    <p class="mdl-typography--caption" style="margin-top: .5em"></p>
                    <p>Genome-scale metabolic models have been used extensively to investigate alterations in cellular metabolism. The accuracy of these models to represent cellular metabolism in specific conditions has been improved by constraining the model with omics data sources. However, few practical methods for integrating metabolomics data with other omics data sources into genome-scale models of metabolism have been developed.</p>
                </div>
                <div class="mdl-card__actions mdl-card--border">
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://doi.org/10.1093/bioinformatics/btt493">Full Article</a>
                    <a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect mdl-color-text--accent"
                        href="https://www.ncbi.nlm.nih.gov/pubmed/23975765">PubMed</a>
                </div>
            </div>
        </div>
        
        <div class="mdl-cell mdl-cell--12-col">
            <h6>Total of 4 publications. 😄</h6>
        </div>
    </div>
    
</div>
	</main>
</div>
</body>
</html>

